// react-country-flags/dist/es/FlagCm.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagCm = function FlagCm(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 90 60"
  }, props), React.createElement("path", {
    fill: "#007a5e",
    d: "M0 0h30v60H0z"
  }), React.createElement("path", {
    fill: "#ce1126",
    d: "M30 0h30v60H30z"
  }), React.createElement("path", {
    fill: "#fcd116",
    d: "M60 0h30v60H60z"
  }), React.createElement("g", {
    transform: "translate(45 30)",
    fill: "#fcd116"
  }, React.createElement("g", {
    id: "sccbc82c5"
  }, React.createElement("path", {
    id: "sa455f754",
    d: "M0-8v8h4z",
    transform: "rotate(18 0 -8)"
  }), React.createElement("use", {
    xlinkHref: "#sa455f754",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#sccbc82c5",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#sccbc82c5",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#sccbc82c5",
    transform: "rotate(216)"
  }), React.createElement("use", {
    xlinkHref: "#sccbc82c5",
    transform: "rotate(288)"
  })));
};

export default FlagCm;
