// react-country-flags/dist/es/FlagUm.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagUm = function FlagUm(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 7410 3900"
  }, props), React.createElement("path", {
    fill: "#b22234",
    d: "M0 0h7410v3900H0z"
  }), React.createElement("path", {
    d: "M0 450h7410m0 600H0m0 600h7410m0 600H0m0 600h7410m0 600H0",
    stroke: "#fff",
    strokeWidth: 300
  }), React.createElement("path", {
    fill: "#3c3b6e",
    d: "M0 0h2964v2100H0z"
  }), React.createElement("g", {
    fill: "#fff"
  }, React.createElement("g", {
    id: "sd492708b"
  }, React.createElement("g", {
    id: "sb046aaaf"
  }, React.createElement("g", {
    id: "s320a5d8f"
  }, React.createElement("g", {
    id: "s67032ec6"
  }, React.createElement("path", {
    id: "s23dcff17",
    d: "M247 90l70.534 217.082-184.66-134.164h228.253L176.466 307.082z"
  }), React.createElement("use", {
    xlinkHref: "#s23dcff17",
    y: 420
  }), React.createElement("use", {
    xlinkHref: "#s23dcff17",
    y: 840
  }), React.createElement("use", {
    xlinkHref: "#s23dcff17",
    y: 1260
  })), React.createElement("use", {
    xlinkHref: "#s23dcff17",
    y: 1680
  })), React.createElement("use", {
    xlinkHref: "#s67032ec6",
    x: 247,
    y: 210
  })), React.createElement("use", {
    xlinkHref: "#sb046aaaf",
    x: 494
  })), React.createElement("use", {
    xlinkHref: "#sd492708b",
    x: 988
  }), React.createElement("use", {
    xlinkHref: "#sb046aaaf",
    x: 1976
  }), React.createElement("use", {
    xlinkHref: "#s320a5d8f",
    x: 2470
  })));
};

export default FlagUm;
