// react-country-flags/dist/es/FlagSn.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagSn = function FlagSn(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 900 600"
  }, props), React.createElement("path", {
    fill: "#00853f",
    d: "M0 0h300v600H0z"
  }), React.createElement("path", {
    fill: "#fdef42",
    d: "M300 0h300v600H300z"
  }), React.createElement("path", {
    fill: "#e31b23",
    d: "M600 0h300v600H600z"
  }), React.createElement("g", {
    transform: "translate(450 300)",
    fill: "#00853f"
  }, React.createElement("g", {
    id: "s0ba0da37"
  }, React.createElement("path", {
    id: "sdc6318e3",
    d: "M0-100V0h50z",
    transform: "rotate(18 0 -100)"
  }), React.createElement("use", {
    xlinkHref: "#sdc6318e3",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#s0ba0da37",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#s0ba0da37",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#s0ba0da37",
    transform: "rotate(216)"
  }), React.createElement("use", {
    xlinkHref: "#s0ba0da37",
    transform: "rotate(288)"
  })));
};

export default FlagSn;
