// react-country-flags/dist/es/FlagWs.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagWs = function FlagWs(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 2880 1440"
  }, props), React.createElement("defs", null, React.createElement("path", {
    id: "s3bed8bf4",
    d: "M0 0l.309.951L-.5.363h1l-.809.588z"
  })), React.createElement("path", {
    fill: "#ce1126",
    d: "M0 0h2880v1440H0z"
  }), React.createElement("path", {
    fill: "#002b7f",
    d: "M0 0h1440v720H0z"
  }), React.createElement("g", {
    fill: "#fff"
  }, React.createElement("use", {
    xlinkHref: "#s3bed8bf4",
    transform: "matrix(160 0 0 160 720 30)"
  }), React.createElement("use", {
    xlinkHref: "#s3bed8bf4",
    transform: "matrix(160 0 0 160 500 210)"
  }), React.createElement("use", {
    xlinkHref: "#s3bed8bf4",
    transform: "matrix(150 0 0 150 945 180)"
  }), React.createElement("use", {
    xlinkHref: "#s3bed8bf4",
    transform: "matrix(100 0 0 100 830 364.894)"
  }), React.createElement("use", {
    xlinkHref: "#s3bed8bf4",
    transform: "matrix(200 0 0 200 720 499.789)"
  })));
};

export default FlagWs;
