// react-country-flags/dist/es/FlagZa.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagZa = function FlagZa(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 9 6"
  }, props), React.createElement("clipPath", {
    id: "s62718bce"
  }, React.createElement("path", {
    d: "M0 0l4.5 3L0 6",
    id: "s55eec673"
  })), React.createElement("clipPath", {
    id: "se7780f5a"
  }, React.createElement("path", {
    d: "M0 0h9v6H0z"
  })), React.createElement("g", {
    clipPath: "url(#se7780f5a)"
  }, React.createElement("path", {
    d: "M0 0v6h9V0z",
    fill: "#002395"
  }), React.createElement("path", {
    d: "M0 0v3h9V0z",
    fill: "#de3831"
  }), React.createElement("g", {
    strokeWidth: 2,
    stroke: "#fff"
  }, React.createElement("path", {
    d: "M0 0l4.5 3L0 6m4.5-3H9",
    id: "s549537da"
  }), React.createElement("use", {
    xlinkHref: "#s55eec673",
    stroke: "#ffb612",
    clipPath: "url(#s62718bce)"
  })), React.createElement("use", {
    xlinkHref: "#s549537da",
    fill: "none",
    stroke: "#007a4d",
    strokeWidth: 1.2
  })));
};

export default FlagZa;
