// react-country-flags/dist/es/FlagAg.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagAg = function FlagAg(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 138 92"
  }, props), React.createElement("path", {
    fill: "#fff",
    d: "M0 0h138v92H0z"
  }), React.createElement("path", {
    d: "M0 0h138v36H0z"
  }), React.createElement("g", {
    transform: "translate(69 36)"
  }, React.createElement("g", {
    id: "sf44718e4"
  }, React.createElement("path", {
    id: "sc66e9395",
    d: "M-30 0L0-5.742V5.742z",
    fill: "#fcd116"
  }), React.createElement("use", {
    xlinkHref: "#sc66e9395",
    transform: "rotate(22.5)"
  }), React.createElement("use", {
    xlinkHref: "#sc66e9395",
    transform: "rotate(45)"
  })), React.createElement("use", {
    xlinkHref: "#sf44718e4",
    transform: "rotate(67.5)"
  }), React.createElement("use", {
    xlinkHref: "#sf44718e4",
    transform: "rotate(135)"
  })), React.createElement("path", {
    fill: "#0072c6",
    d: "M0 36h138v20H0z"
  }), React.createElement("path", {
    d: "M0 0v92h68M138 0v92H68",
    fill: "#ce1126"
  }));
};

export default FlagAg;
