// react-country-flags/dist/es/FlagNz.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagNz = function FlagNz(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 1200 600"
  }, props), React.createElement("defs", null, React.createElement("clipPath", {
    id: "sa3487802"
  }, React.createElement("path", {
    d: "M0 0h600v300H0z"
  })), React.createElement("clipPath", {
    id: "sa841c70d"
  }, React.createElement("path", {
    d: "M0 0l300 150H0zm300 0h300L300 150zm0 150h300v150zm0 0v150H0z"
  })), React.createElement("g", {
    id: "s4ed22915"
  }, React.createElement("g", {
    id: "s7f67afd8"
  }, React.createElement("path", {
    d: "M0 0v.5L1 0z",
    transform: "translate(0 -.325)"
  }), React.createElement("path", {
    d: "M0 0v-.5L1 0z",
    transform: "rotate(-36 .5 -.162)"
  })), React.createElement("use", {
    xlinkHref: "#s7f67afd8",
    transform: "scale(-1 1)"
  }), React.createElement("use", {
    xlinkHref: "#s7f67afd8",
    transform: "rotate(72 0 0)"
  }), React.createElement("use", {
    xlinkHref: "#s7f67afd8",
    transform: "rotate(-72 0 0)"
  }), React.createElement("use", {
    xlinkHref: "#s7f67afd8",
    transform: "scale(-1 1) rotate(72)"
  }))), React.createElement("path", {
    fill: "#00247d",
    d: "M0 0h1200v600H0z"
  }), React.createElement("path", {
    stroke: "#fff",
    d: "M0 0l600 300M0 300L600 0",
    strokeWidth: 60,
    clipPath: "url(#sa3487802)"
  }), React.createElement("path", {
    stroke: "#cc142b",
    d: "M0 0l600 300M0 300L600 0",
    strokeWidth: 40,
    clipPath: "url(#sa841c70d)"
  }), React.createElement("path", {
    stroke: "#fff",
    d: "M300 0v300M0 150h600",
    strokeWidth: 100,
    clipPath: "url(#sa3487802)"
  }), React.createElement("path", {
    stroke: "#cc142b",
    d: "M300 0v300M0 150h600",
    strokeWidth: 60,
    clipPath: "url(#sa3487802)"
  }), React.createElement("g", null, React.createElement("use", {
    xlinkHref: "#s4ed22915",
    fill: "#fff",
    transform: "matrix(45.4 0 0 45.4 900 120)"
  }), React.createElement("use", {
    xlinkHref: "#s4ed22915",
    fill: "#cc142b",
    transform: "matrix(30 0 0 30 900 120)"
  }), React.createElement("g", {
    transform: "rotate(82 900 240)"
  }, React.createElement("use", {
    xlinkHref: "#s4ed22915",
    fill: "#fff",
    transform: "rotate(-82 519.022 -457.666) scale(40.4)"
  }), React.createElement("use", {
    xlinkHref: "#s4ed22915",
    fill: "#cc142b",
    transform: "rotate(-82 519.022 -457.666) scale(25)"
  })), React.createElement("g", {
    transform: "rotate(82 900 240)"
  }, React.createElement("use", {
    xlinkHref: "#s4ed22915",
    fill: "#fff",
    transform: "rotate(-82 668.57 -327.666) scale(45.4)"
  }), React.createElement("use", {
    xlinkHref: "#s4ed22915",
    fill: "#cc142b",
    transform: "rotate(-82 668.57 -327.666) scale(30)"
  })), React.createElement("g", null, React.createElement("use", {
    xlinkHref: "#s4ed22915",
    fill: "#fff",
    transform: "matrix(50.4 0 0 50.4 900 480)"
  }), React.createElement("use", {
    xlinkHref: "#s4ed22915",
    fill: "#cc142b",
    transform: "matrix(35 0 0 35 900 480)"
  }))));
};

export default FlagNz;
