// react-country-flags/dist/es/FlagCw.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagCw = function FlagCw(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 54 36"
  }, props), React.createElement("path", {
    fill: "#002b7f",
    d: "M0 0h54v36H0z"
  }), React.createElement("path", {
    d: "M0 22.5h54V27H0z",
    fill: "#f9e814"
  }), React.createElement("g", {
    fill: "#fff",
    id: "s493bbfcf"
  }, React.createElement("g", {
    id: "s6b1ec82d"
  }, React.createElement("g", {
    id: "s4581c9ca"
  }, React.createElement("path", {
    d: "M12 8v4h2z",
    transform: "rotate(18 12 8)",
    id: "sbbe6da30"
  }), React.createElement("use", {
    xlinkHref: "#sbbe6da30",
    x: -24,
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#s4581c9ca",
    transform: "rotate(72 12 12)"
  })), React.createElement("use", {
    xlinkHref: "#s4581c9ca",
    transform: "rotate(-72 12 12)"
  }), React.createElement("use", {
    xlinkHref: "#s6b1ec82d",
    transform: "rotate(144 12 12)"
  })), React.createElement("use", {
    xlinkHref: "#s493bbfcf",
    x: -4,
    y: -4,
    transform: "scale(.75)"
  }));
};

export default FlagCw;
