// react-country-flags/dist/es/FlagSt.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagSt = function FlagSt(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 2800 1400"
  }, props), React.createElement("path", {
    fill: "#12ad2b",
    d: "M0 0h2800v1400H0z"
  }), React.createElement("path", {
    fill: "#ffce00",
    d: "M0 400h2800v600H0z"
  }), React.createElement("path", {
    d: "M0 0v1400l700-700",
    fill: "#d21034"
  }), React.createElement("g", {
    id: "s90568f6a",
    transform: "translate(1400 700)",
    fill: "#000"
  }, React.createElement("g", {
    id: "s8c3f5a9e"
  }, React.createElement("path", {
    id: "s2f1447cc",
    d: "M0-200V0h100",
    transform: "rotate(18 0 -200)"
  }), React.createElement("use", {
    xlinkHref: "#s2f1447cc",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#s8c3f5a9e",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#s8c3f5a9e",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#s8c3f5a9e",
    transform: "rotate(216)"
  }), React.createElement("use", {
    xlinkHref: "#s8c3f5a9e",
    transform: "rotate(288)"
  })), React.createElement("use", {
    xlinkHref: "#s90568f6a",
    x: 700
  }));
};

export default FlagSt;
