// react-country-flags/dist/es/FlagFm.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagFm = function FlagFm(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "-19 -10 38 20"
  }, props), React.createElement("path", {
    fill: "#75b2dd",
    d: "M-19-10h38v20h-38z"
  }), React.createElement("g", {
    id: "s17ec4d17",
    transform: "translate(0 -6)",
    fill: "#fff"
  }, React.createElement("g", {
    id: "sf00899f4"
  }, React.createElement("path", {
    id: "s0cf11e6f",
    d: "M0-2v2h1",
    transform: "rotate(18 0 -2)"
  }), React.createElement("use", {
    xlinkHref: "#s0cf11e6f",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#sf00899f4",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#sf00899f4",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#sf00899f4",
    transform: "rotate(216)"
  }), React.createElement("use", {
    xlinkHref: "#sf00899f4",
    transform: "rotate(288)"
  })), React.createElement("use", {
    xlinkHref: "#s17ec4d17",
    transform: "rotate(90)"
  }), React.createElement("use", {
    xlinkHref: "#s17ec4d17",
    transform: "rotate(180)"
  }), React.createElement("use", {
    xlinkHref: "#s17ec4d17",
    transform: "rotate(270)"
  }));
};

export default FlagFm;
