// react-country-flags/dist/es/FlagNa.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagNa = function FlagNa(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 90 60"
  }, props), React.createElement("svg", {
    viewBox: "0 0 90 60"
  }, React.createElement("path", {
    d: "M90 0H0v60z",
    fill: "#003580"
  }), React.createElement("path", {
    d: "M0 60h90V0z",
    fill: "#009543"
  }), React.createElement("path", {
    d: "M0 60L90 0",
    stroke: "#fff",
    strokeWidth: 20
  }), React.createElement("path", {
    d: "M0 60L90 0",
    stroke: "#d21034",
    strokeWidth: 15
  }), React.createElement("g", {
    fill: "#ffce00",
    transform: "translate(18 16.341)"
  }, React.createElement("g", {
    id: "s5c6eab9b"
  }, React.createElement("path", {
    id: "sb5936373",
    d: "M0 10l1.553-4.204h-3.106z"
  }), React.createElement("use", {
    xlinkHref: "#sb5936373",
    transform: "rotate(90)"
  }), React.createElement("use", {
    xlinkHref: "#sb5936373",
    transform: "rotate(180)"
  }), React.createElement("use", {
    xlinkHref: "#sb5936373",
    transform: "rotate(270)"
  })), React.createElement("use", {
    xlinkHref: "#s5c6eab9b",
    transform: "rotate(30)"
  }), React.createElement("use", {
    xlinkHref: "#s5c6eab9b",
    transform: "rotate(60)"
  }), React.createElement("circle", {
    r: 5.5,
    stroke: "#003580"
  }))));
};

export default FlagNa;
