// react-country-flags/dist/es/FlagPr.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagPr = function FlagPr(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 45 30"
  }, props), React.createElement("path", {
    fill: "#ED0000",
    d: "M0 0h45v30H0z"
  }), React.createElement("path", {
    stroke: "#FFF",
    strokeWidth: 6,
    d: "M0 9h45M0 21h45"
  }), React.createElement("path", {
    fill: "#0050F0",
    d: "M0 0l25.98 15L0 30z"
  }), React.createElement("g", {
    fill: "#FFF",
    transform: "matrix(5 0 0 5 8.66 15)"
  }, React.createElement("g", {
    id: "s53d7d484"
  }, React.createElement("path", {
    id: "s80f072d4",
    transform: "rotate(18 3.157 -.5)",
    d: "M0 0v1h.5z"
  }), React.createElement("use", {
    xlinkHref: "#s80f072d4",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#s53d7d484",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#s53d7d484",
    transform: "rotate(-72)"
  }), React.createElement("use", {
    xlinkHref: "#s53d7d484",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#s53d7d484",
    transform: "rotate(-144)"
  })));
};

export default FlagPr;
