// react-country-flags/dist/es/FlagBi.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagBi = function FlagBi(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 500 300"
  }, props), React.createElement("defs", null, React.createElement("g", {
    id: "se6fdf592",
    fill: "#1eb53a"
  }, React.createElement("g", {
    id: "s0c0c4fed"
  }, React.createElement("g", {
    id: "sa8e2a64b"
  }, React.createElement("g", {
    id: "s25c8ce82"
  }, React.createElement("path", {
    id: "sc56da287",
    d: "M0-20V0h20",
    transform: "rotate(30 0 -20)"
  }), React.createElement("use", {
    xlinkHref: "#sc56da287",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#s25c8ce82",
    transform: "rotate(120)"
  }), React.createElement("use", {
    xlinkHref: "#s25c8ce82",
    transform: "rotate(240)"
  })), React.createElement("use", {
    xlinkHref: "#sa8e2a64b",
    transform: "rotate(180)"
  })), React.createElement("use", {
    xlinkHref: "#s0c0c4fed",
    fill: "#ce1126",
    transform: "scale(.82)"
  })), React.createElement("use", {
    id: "se40402f7",
    xlinkHref: "#se6fdf592",
    x: 250,
    y: 106
  })), React.createElement("path", {
    d: "M0 0h500L0 300h500z",
    fill: "#ce1126"
  }), React.createElement("path", {
    d: "M0 0v300L500 0v300z",
    fill: "#1eb53a"
  }), React.createElement("path", {
    d: "M0 0l500 300m0-300L0 300",
    stroke: "#fff",
    strokeWidth: 40
  }), React.createElement("circle", {
    cx: 250,
    cy: 150,
    r: 85,
    fill: "#fff"
  }), React.createElement("use", {
    xlinkHref: "#se40402f7"
  }), React.createElement("use", {
    xlinkHref: "#se40402f7",
    transform: "rotate(120 250 150)"
  }), React.createElement("use", {
    xlinkHref: "#se40402f7",
    transform: "rotate(240 250 150)"
  }));
};

export default FlagBi;
