// react-country-flags/dist/es/FlagVn.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagVn = function FlagVn(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "-15 -10 30 20"
  }, props), React.createElement("path", {
    fill: "#DA251d",
    d: "M-20-15h40v30h-40z"
  }), React.createElement("g", {
    id: "sbd7c3fad",
    transform: "translate(0 -6)"
  }, React.createElement("path", {
    id: "sf915d7b8",
    fill: "#FF0",
    transform: "rotate(18)",
    d: "M0 0v6h4"
  }), React.createElement("use", {
    xlinkHref: "#sf915d7b8",
    transform: "scale(-1 1)"
  })), React.createElement("g", {
    id: "sca97a1a1",
    transform: "rotate(72)"
  }, React.createElement("use", {
    xlinkHref: "#sbd7c3fad"
  }), React.createElement("use", {
    xlinkHref: "#sbd7c3fad",
    transform: "rotate(72)"
  })), React.createElement("use", {
    xlinkHref: "#sca97a1a1",
    transform: "scale(-1 1)"
  }));
};

export default FlagVn;
