// react-country-flags/dist/es/FlagKg.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagKg = function FlagKg(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 250 150"
  }, props), React.createElement("clipPath", {
    id: "s87332abe"
  }, React.createElement("circle", {
    r: 25
  })), React.createElement("path", {
    fill: "#e8112d",
    d: "M0 0h250v150H0z"
  }), React.createElement("g", {
    transform: "translate(125 75)"
  }, React.createElement("g", {
    id: "sa4828ec8",
    transform: "scale(.25)"
  }, React.createElement("g", {
    id: "s3bcf823f"
  }, React.createElement("path", {
    id: "sf2d4085e",
    d: "M-8.4 108C0 130-18 164 0 180c-6-24 14-50 8.4-72s-25.2-22-16.8 0z",
    fill: "#ffef00"
  }), React.createElement("use", {
    xlinkHref: "#sf2d4085e",
    transform: "rotate(90)"
  }), React.createElement("use", {
    xlinkHref: "#sf2d4085e",
    transform: "rotate(180)"
  }), React.createElement("use", {
    xlinkHref: "#sf2d4085e",
    transform: "rotate(270)"
  })), React.createElement("use", {
    xlinkHref: "#s3bcf823f",
    transform: "rotate(18)"
  }), React.createElement("use", {
    xlinkHref: "#s3bcf823f",
    transform: "rotate(36)"
  }), React.createElement("use", {
    xlinkHref: "#s3bcf823f",
    transform: "rotate(54)"
  }), React.createElement("use", {
    xlinkHref: "#s3bcf823f",
    transform: "rotate(72)"
  })), React.createElement("use", {
    xlinkHref: "#sa4828ec8",
    transform: "rotate(9)"
  }), React.createElement("circle", {
    r: 27,
    fill: "#e8112d"
  }), React.createElement("circle", {
    cy: -1.5,
    r: 22.5,
    fill: "#ffef00"
  }), React.createElement("g", {
    id: "s89fbd3d2",
    fill: "#e8112d",
    clipPath: "url(#s87332abe)"
  }, React.createElement("path", {
    d: "M-23.055 18a37.755 37.755 0 1 1 75.51 0h.305a36.26 36.26 0 1 0-72.52 0z"
  }), React.createElement("path", {
    d: "M-17.601 18A35.301 35.301 0 1 1 53 18h.424a33.925 33.925 0 1 0-67.85 0z"
  }), React.createElement("path", {
    d: "M20.7-15.052a33.052 33.052 0 1 0 0 66.104l1.8-1.232a31.82 31.82 0 1 1 0-63.64z"
  })), React.createElement("use", {
    xlinkHref: "#s89fbd3d2",
    transform: "scale(-1 1)"
  })));
};

export default FlagKg;
