// react-country-flags/dist/es/FlagHn.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagHn = function FlagHn(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 72 36"
  }, props), React.createElement("path", {
    fill: "#0073cf",
    d: "M0 0h72v36H0z"
  }), React.createElement("path", {
    fill: "#fff",
    d: "M0 12h72v12H0z"
  }), React.createElement("g", {
    id: "se30fab8f",
    transform: "matrix(2 0 0 2 36 18)",
    fill: "#0073cf"
  }, React.createElement("g", {
    id: "s1efae5c5"
  }, React.createElement("path", {
    id: "s0725a184",
    transform: "rotate(18 3.157 -.5)",
    d: "M0 0v1h.5z"
  }), React.createElement("use", {
    xlinkHref: "#s0725a184",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#s1efae5c5",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#s1efae5c5",
    transform: "rotate(-72)"
  }), React.createElement("use", {
    xlinkHref: "#s1efae5c5",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#s1efae5c5",
    transform: "rotate(-144)"
  })), React.createElement("use", {
    xlinkHref: "#se30fab8f",
    transform: "translate(10 -3.2)"
  }), React.createElement("use", {
    xlinkHref: "#se30fab8f",
    transform: "translate(10 2.8)"
  }), React.createElement("use", {
    xlinkHref: "#se30fab8f",
    transform: "translate(-10 -3.2)"
  }), React.createElement("use", {
    xlinkHref: "#se30fab8f",
    transform: "translate(-10 2.8)"
  }));
};

export default FlagHn;
