// react-country-flags/dist/es/FlagKn.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagKn = function FlagKn(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 750 500"
  }, props), React.createElement("svg", {
    viewBox: "0 0 750 500"
  }, React.createElement("path", {
    d: "M750 0H0v500",
    fill: "#009e49"
  }), React.createElement("path", {
    d: "M0 500h750V0",
    fill: "#ce1126"
  }), React.createElement("path", {
    d: "M0 500L750 0",
    stroke: "#fcd116",
    strokeWidth: 210
  }), React.createElement("path", {
    d: "M0 500L750 0",
    stroke: "#000",
    strokeWidth: 150
  }), React.createElement("g", {
    id: "sfc7af856",
    transform: "rotate(-33.69 514.716 -777.095)",
    fill: "#fff"
  }, React.createElement("g", {
    id: "s949522d6"
  }, React.createElement("path", {
    id: "s6930e572",
    d: "M0-70V0h35",
    transform: "rotate(18 0 -70)"
  }), React.createElement("use", {
    xlinkHref: "#s6930e572",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#s949522d6",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#s949522d6",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#s949522d6",
    transform: "rotate(216)"
  }), React.createElement("use", {
    xlinkHref: "#s949522d6",
    transform: "rotate(288)"
  })), React.createElement("use", {
    xlinkHref: "#sfc7af856",
    transform: "translate(-285 190)"
  })));
};

export default FlagKn;
