// react-country-flags/dist/es/FlagTg.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagTg = function FlagTg(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 809 500"
  }, props), React.createElement("path", {
    fill: "#006a4e",
    d: "M0 0h809v500H0z"
  }), React.createElement("path", {
    fill: "#ffce00",
    d: "M0 100h809v100H0zM0 300h809v100H0z"
  }), React.createElement("path", {
    fill: "#d21034",
    d: "M0 0h300v300H0z"
  }), React.createElement("g", {
    transform: "translate(150 150)",
    fill: "#fff"
  }, React.createElement("g", {
    id: "s07b6308c"
  }, React.createElement("path", {
    id: "s2c3db29c",
    d: "M0-95V0h50",
    transform: "rotate(18 0 -95)"
  }), React.createElement("use", {
    xlinkHref: "#s2c3db29c",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#s07b6308c",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#s07b6308c",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#s07b6308c",
    transform: "rotate(216)"
  }), React.createElement("use", {
    xlinkHref: "#s07b6308c",
    transform: "rotate(288)"
  })));
};

export default FlagTg;
