// react-country-flags/dist/es/FlagTv.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagTv = function FlagTv(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 1200 600"
  }, props), React.createElement("defs", null, React.createElement("clipPath", {
    id: "sc2e6d055"
  }, React.createElement("path", {
    d: "M0 0v150h700v150H600zm0 300v100h300V0h300z"
  })), React.createElement("path", {
    id: "s28337b13",
    fill: "#ffce00",
    d: "M0-50.08l29.437 90.596-77.067-55.992h95.26l-77.067 55.993z"
  }), React.createElement("use", {
    id: "se6d353a4",
    xlinkHref: "#s28337b13",
    transform: "rotate(36)"
  })), React.createElement("path", {
    fill: "#00247d",
    d: "M0 0h1200v600H0z"
  }), React.createElement("path", {
    d: "M0 0l600 300M0 300L600 0",
    stroke: "#fff",
    strokeWidth: 60
  }), React.createElement("path", {
    d: "M0 0l600 300M0 300L600 0",
    stroke: "#cf142b",
    strokeWidth: 40,
    clipPath: "url(#sc2e6d055)"
  }), React.createElement("path", {
    d: "M300 0v400M0 150h700",
    stroke: "#fff",
    strokeWidth: 100
  }), React.createElement("path", {
    d: "M300 0v400M0 150h700",
    stroke: "#cf142b",
    strokeWidth: 60
  }), React.createElement("path", {
    d: "M0 300h600V0h600v600H0z",
    fill: "#5b97b1"
  }), React.createElement("use", {
    xlinkHref: "#s28337b13",
    x: 645.34,
    y: 540.994
  }), React.createElement("use", {
    xlinkHref: "#se6d353a4",
    x: 788.026,
    y: 504.516
  }), React.createElement("use", {
    xlinkHref: "#se6d353a4",
    x: 788.026,
    y: 382.532
  }), React.createElement("use", {
    xlinkHref: "#se6d353a4",
    x: 914.03,
    y: 473.684
  }), React.createElement("use", {
    xlinkHref: "#se6d353a4",
    x: 914.03,
    y: 208.271
  }), React.createElement("use", {
    xlinkHref: "#s28337b13",
    x: 996.442,
    y: 390.046
  }), React.createElement("use", {
    xlinkHref: "#se6d353a4",
    x: 1019.928,
    y: 172.078
  }), React.createElement("use", {
    xlinkHref: "#s28337b13",
    x: 1102.836,
    y: 321.554
  }), React.createElement("use", {
    xlinkHref: "#s28337b13",
    x: 1102.836,
    y: 87.486
  }));
};

export default FlagTv;
