// react-country-flags/dist/es/FlagMm.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagMm = function FlagMm(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 18 12"
  }, props), React.createElement("path", {
    fill: "#FECB00",
    d: "M0 0h18v6H0z"
  }), React.createElement("path", {
    fill: "#EA2839",
    d: "M0 6h18v6H0z"
  }), React.createElement("path", {
    fill: "#34B233",
    d: "M0 4h18v4H0z"
  }), React.createElement("g", {
    transform: "translate(9 6.422) scale(4.422)"
  }, React.createElement("path", {
    id: "s363654a1",
    fill: "#FFF",
    d: "M-.325 0L0-1l.325 1z"
  }), React.createElement("use", {
    xlinkHref: "#s363654a1",
    transform: "rotate(-144)"
  }), React.createElement("use", {
    xlinkHref: "#s363654a1",
    transform: "rotate(-72)"
  }), React.createElement("use", {
    xlinkHref: "#s363654a1",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#s363654a1",
    transform: "rotate(144)"
  })));
};

export default FlagMm;
