// react-country-flags/dist/es/FlagSy.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagSy = function FlagSy(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 180 120"
  }, props), React.createElement("path", {
    d: "M0 0h180v120H0z"
  }), React.createElement("path", {
    fill: "#fff",
    d: "M0 0h180v80H0z"
  }), React.createElement("path", {
    fill: "#ce1126",
    d: "M0 0h180v40H0z"
  }), React.createElement("g", {
    id: "s3f41d733",
    fill: "#007a3d"
  }, React.createElement("g", {
    id: "see16fe3e"
  }, React.createElement("g", {
    id: "s2bec85f8"
  }, React.createElement("path", {
    d: "M54 47v13h8",
    transform: "rotate(18 54 47)",
    id: "s9ec60cac"
  }), React.createElement("use", {
    xlinkHref: "#s9ec60cac",
    x: -108,
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#s2bec85f8",
    transform: "rotate(72 54 60)"
  })), React.createElement("use", {
    xlinkHref: "#s2bec85f8",
    transform: "rotate(-72 54 60)"
  }), React.createElement("use", {
    xlinkHref: "#see16fe3e",
    transform: "rotate(144 54 60)"
  })), React.createElement("use", {
    xlinkHref: "#s3f41d733",
    x: 72
  }));
};

export default FlagSy;
