// react-country-flags/dist/es/FlagSb.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagSb = function FlagSb(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 800 400"
  }, props), React.createElement("svg", {
    viewBox: "0 0 800 400"
  }, React.createElement("path", {
    d: "M0 400V0h800z",
    fill: "#0051ba"
  }), React.createElement("path", {
    d: "M0 400h800V0z",
    fill: "#215b33"
  }), React.createElement("path", {
    d: "M0 400L800 0",
    stroke: "#fcd116",
    strokeWidth: 36
  }), React.createElement("g", {
    transform: "translate(140 120)"
  }, React.createElement("g", {
    id: "sa931cb16",
    fill: "#fff"
  }, React.createElement("g", {
    id: "s0ff378c8"
  }, React.createElement("g", {
    id: "s7f7678ea"
  }, React.createElement("path", {
    id: "sd3ee06bb",
    d: "M0-40V0h20z",
    transform: "rotate(18 0 -40)"
  }), React.createElement("use", {
    xlinkHref: "#sd3ee06bb",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#s7f7678ea",
    transform: "rotate(72)"
  })), React.createElement("use", {
    xlinkHref: "#s7f7678ea",
    transform: "rotate(-72)"
  }), React.createElement("use", {
    xlinkHref: "#s0ff378c8",
    transform: "rotate(144)"
  })), React.createElement("g", {
    id: "s9554276b",
    transform: "rotate(40.6)"
  }, React.createElement("use", {
    id: "s89fe461e",
    xlinkHref: "#sa931cb16",
    x: -104,
    transform: "rotate(-40.6 -104 0)"
  }), React.createElement("use", {
    xlinkHref: "#s89fe461e",
    x: 208
  })), React.createElement("use", {
    xlinkHref: "#s9554276b",
    transform: "scale(-1 1)"
  }))));
};

export default FlagSb;
