// react-country-flags/dist/es/FlagCl.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagCl = function FlagCl(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "-4 -4 24 16"
  }, props), React.createElement("path", {
    d: "M20 12H-4V-4h24z",
    fill: "#d52b1e"
  }), React.createElement("path", {
    d: "M4 4h16v-8H-4z",
    fill: "#fff"
  }), React.createElement("path", {
    d: "M4 4h-8v-8h8z",
    fill: "#0039a6"
  }), React.createElement("g", {
    id: "s39d4d252"
  }, React.createElement("g", {
    id: "sd7c065a0"
  }, React.createElement("path", {
    d: "M0-2v2h1z",
    fill: "#fff",
    transform: "rotate(18 0 -2)",
    id: "s2b94775c"
  }), React.createElement("use", {
    xlinkHref: "#s2b94775c",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#sd7c065a0",
    transform: "rotate(72)"
  })), React.createElement("use", {
    xlinkHref: "#sd7c065a0",
    transform: "rotate(-72)"
  }), React.createElement("use", {
    xlinkHref: "#s39d4d252",
    transform: "rotate(144)"
  }));
};

export default FlagCl;
