// react-country-flags/dist/es/FlagEt.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagEt = function FlagEt(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 720 360"
  }, props), React.createElement("path", {
    fill: "#da121a",
    d: "M0 0h720v360H0z"
  }), React.createElement("path", {
    fill: "#fcdd09",
    d: "M0 0h720v240H0z"
  }), React.createElement("path", {
    fill: "#078930",
    d: "M0 0h720v120H0z"
  }), React.createElement("g", {
    transform: "translate(360 180)"
  }, React.createElement("circle", {
    fill: "#0f47af",
    r: 120
  }), React.createElement("g", {
    id: "se519047c"
  }, React.createElement("path", {
    fill: "#fcdd09",
    d: "M0-96l-4.206 12.944 17.348 53.39h-23.13l-2.599 8h74.163l11.011-8H21.553z"
  }), React.createElement("path", {
    stroke: "#fcdd09",
    strokeWidth: 4,
    d: "M25.863-35.597l30.564-42.069"
  })), React.createElement("use", {
    xlinkHref: "#se519047c",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#se519047c",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#se519047c",
    transform: "rotate(216)"
  }), React.createElement("use", {
    xlinkHref: "#se519047c",
    transform: "rotate(288)"
  })));
};

export default FlagEt;
