// react-country-flags/dist/es/FlagBf.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagBf = function FlagBf(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 900 600"
  }, props), React.createElement("path", {
    fill: "#009e49",
    d: "M0 0h900v600H0z"
  }), React.createElement("path", {
    fill: "#ef2b2d",
    d: "M0 0h900v300H0z"
  }), React.createElement("g", {
    transform: "translate(450 300)",
    fill: "#fcd116"
  }, React.createElement("g", {
    id: "s71e1f406"
  }, React.createElement("path", {
    id: "s7d36ca28",
    d: "M0-100V0h50",
    transform: "rotate(18 0 -100)"
  }), React.createElement("use", {
    xlinkHref: "#s7d36ca28",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#s71e1f406",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#s71e1f406",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#s71e1f406",
    transform: "rotate(216)"
  }), React.createElement("use", {
    xlinkHref: "#s71e1f406",
    transform: "rotate(288)"
  })));
};

export default FlagBf;
