// react-country-flags/dist/es/FlagCk.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagCk = function FlagCk(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 600 300"
  }, props), React.createElement("clipPath", {
    id: "s85203e2b"
  }, React.createElement("path", {
    d: "M0 0v75h350v75h-50zm300 0H150v200H0v-50z"
  })), React.createElement("path", {
    fill: "#00247d",
    d: "M0 0h600v300H0z"
  }), React.createElement("path", {
    d: "M0 0l300 150m0-150L0 150",
    stroke: "#fff",
    strokeWidth: 30
  }), React.createElement("path", {
    d: "M0 0l300 150m0-150L0 150",
    stroke: "#cf142b",
    strokeWidth: 20,
    clipPath: "url(#s85203e2b)"
  }), React.createElement("path", {
    d: "M150 0v200M0 75h350",
    stroke: "#fff",
    strokeWidth: 50
  }), React.createElement("path", {
    d: "M150 0v200M0 75h350",
    stroke: "#cf142b",
    strokeWidth: 30
  }), React.createElement("path", {
    d: "M300 0v150H0v150h600V0z",
    fill: "#00247d"
  }), React.createElement("g", {
    transform: "translate(450 150)"
  }, React.createElement("g", {
    id: "s49ffa104",
    transform: "translate(0 -102)",
    fill: "#fff"
  }, React.createElement("g", {
    id: "s39779b07"
  }, React.createElement("path", {
    id: "sc43daf61",
    d: "M0-22.5V0h12",
    transform: "rotate(18 0 -22.5)"
  }), React.createElement("use", {
    xlinkHref: "#sc43daf61",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#s39779b07",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#s39779b07",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#s39779b07",
    transform: "rotate(216)"
  }), React.createElement("use", {
    xlinkHref: "#s39779b07",
    transform: "rotate(288)"
  })), React.createElement("g", {
    id: "s78382fbe"
  }, React.createElement("use", {
    xlinkHref: "#s49ffa104",
    transform: "rotate(24)"
  }), React.createElement("use", {
    xlinkHref: "#s49ffa104",
    transform: "rotate(48)"
  }), React.createElement("use", {
    xlinkHref: "#s49ffa104",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#s49ffa104",
    transform: "rotate(96)"
  }), React.createElement("use", {
    xlinkHref: "#s49ffa104",
    transform: "rotate(120)"
  }), React.createElement("use", {
    xlinkHref: "#s49ffa104",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#s49ffa104",
    transform: "rotate(168)"
  })), React.createElement("use", {
    xlinkHref: "#s78382fbe",
    transform: "rotate(168)"
  })));
};

export default FlagCk;
