// react-country-flags/dist/es/FlagVe.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagVe = function FlagVe(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 180 120"
  }, props), React.createElement("defs", null, React.createElement("g", {
    id: "s17194599",
    transform: "translate(0 -36)"
  }, React.createElement("g", {
    id: "sb6cb2187"
  }, React.createElement("g", {
    id: "s70610266"
  }, React.createElement("path", {
    d: "M0-5v5h3z",
    fill: "#fff",
    transform: "rotate(18 0 -5)",
    id: "sa8143e40"
  }), React.createElement("use", {
    xlinkHref: "#sa8143e40",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#s70610266",
    transform: "rotate(72)"
  })), React.createElement("use", {
    xlinkHref: "#s70610266",
    transform: "rotate(-72)"
  }), React.createElement("use", {
    xlinkHref: "#sb6cb2187",
    transform: "rotate(144)"
  }))), React.createElement("path", {
    d: "M0 0h180v120H0z",
    fill: "#cf142b"
  }), React.createElement("path", {
    d: "M0 0h180v80H0z",
    fill: "#00247d"
  }), React.createElement("path", {
    d: "M0 0h180v40H0z",
    fill: "#fc0"
  }), React.createElement("g", {
    transform: "translate(90 84)"
  }, React.createElement("g", {
    id: "s5e5470d0"
  }, React.createElement("g", {
    id: "sf6865a85"
  }, React.createElement("use", {
    xlinkHref: "#s17194599",
    transform: "rotate(10)"
  }), React.createElement("use", {
    xlinkHref: "#s17194599",
    transform: "rotate(30)"
  })), React.createElement("use", {
    xlinkHref: "#sf6865a85",
    transform: "rotate(40)"
  })), React.createElement("use", {
    xlinkHref: "#s5e5470d0",
    transform: "rotate(-80)"
  })));
};

export default FlagVe;
