// react-country-flags/dist/es/FlagMh.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagMh = function FlagMh(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 570 300"
  }, props), React.createElement("path", {
    fill: "#003893",
    d: "M0 0h570v300H0z"
  }), React.createElement("path", {
    d: "M0 295.2V288L570 35v87.4z",
    fill: "#fff"
  }), React.createElement("path", {
    d: "M0 290.4v-4.8L570 4.8v58.8z",
    fill: "#dd7500"
  }), React.createElement("g", {
    fill: "#fff",
    transform: "translate(109.536 109.536)"
  }, React.createElement("g", {
    id: "s082e676b"
  }, React.createElement("path", {
    d: "M0-93L5.59 0H-5.59z"
  }), React.createElement("path", {
    id: "se1b5eca9",
    d: "M0-66.588L6.824 0H-6.824z",
    transform: "rotate(15)"
  }), React.createElement("use", {
    xlinkHref: "#se1b5eca9",
    transform: "rotate(15)"
  }), React.createElement("use", {
    xlinkHref: "#se1b5eca9",
    transform: "rotate(30)"
  }), React.createElement("use", {
    xlinkHref: "#se1b5eca9",
    transform: "rotate(45)"
  }), React.createElement("use", {
    xlinkHref: "#se1b5eca9",
    transform: "rotate(60)"
  })), React.createElement("use", {
    xlinkHref: "#s082e676b",
    transform: "rotate(90)"
  }), React.createElement("use", {
    xlinkHref: "#s082e676b",
    transform: "rotate(180)"
  }), React.createElement("use", {
    xlinkHref: "#s082e676b",
    transform: "rotate(270)"
  })));
};

export default FlagMh;
