// react-country-flags/dist/es/FlagGw.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagGw = function FlagGw(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "-2 -3 12 6"
  }, props), React.createElement("path", {
    fill: "#ce1126",
    d: "M-2-3h4v6h-4z"
  }), React.createElement("path", {
    fill: "#fcd116",
    d: "M2-3h8v3H2z"
  }), React.createElement("path", {
    fill: "#009e49",
    d: "M2 0h8v3H2z"
  }), React.createElement("g", {
    id: "sb56d3e06"
  }, React.createElement("path", {
    id: "sfbfd31ce",
    d: "M0-1v1h.5",
    transform: "rotate(18 0 -1)",
    fill: "#000"
  }), React.createElement("use", {
    xlinkHref: "#sfbfd31ce",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#sb56d3e06",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#sb56d3e06",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#sb56d3e06",
    transform: "rotate(216)"
  }), React.createElement("use", {
    xlinkHref: "#sb56d3e06",
    transform: "rotate(288)"
  }));
};

export default FlagGw;
