// react-country-flags/dist/es/FlagEu.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagEu = function FlagEu(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 810 540"
  }, props), React.createElement("defs", null, React.createElement("g", {
    id: "s91bf8332"
  }, React.createElement("g", {
    id: "sf412d9e4"
  }, React.createElement("path", {
    id: "s0f53adc6",
    d: "M0 0v1h.5z",
    transform: "rotate(18 3.157 -.5)"
  }), React.createElement("use", {
    xlinkHref: "#s0f53adc6",
    transform: "scale(-1 1)"
  })), React.createElement("g", {
    id: "sa8efbd1a"
  }, React.createElement("use", {
    xlinkHref: "#sf412d9e4",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#sf412d9e4",
    transform: "rotate(144)"
  })), React.createElement("use", {
    xlinkHref: "#sa8efbd1a",
    transform: "scale(-1 1)"
  }))), React.createElement("path", {
    fill: "#039",
    d: "M0 0h810v540H0z"
  }), React.createElement("g", {
    fill: "#fc0",
    transform: "matrix(30 0 0 30 405 270)"
  }, React.createElement("use", {
    xlinkHref: "#s91bf8332",
    y: -6
  }), React.createElement("use", {
    xlinkHref: "#s91bf8332",
    y: 6
  }), React.createElement("g", {
    id: "s5cea22cb"
  }, React.createElement("use", {
    xlinkHref: "#s91bf8332",
    x: -6
  }), React.createElement("use", {
    xlinkHref: "#s91bf8332",
    transform: "rotate(-144 -2.344 -2.11)"
  }), React.createElement("use", {
    xlinkHref: "#s91bf8332",
    transform: "rotate(144 -2.11 -2.344)"
  }), React.createElement("use", {
    xlinkHref: "#s91bf8332",
    transform: "rotate(72 -4.663 -2.076)"
  }), React.createElement("use", {
    xlinkHref: "#s91bf8332",
    transform: "rotate(72 -5.076 .534)"
  })), React.createElement("use", {
    xlinkHref: "#s5cea22cb",
    transform: "scale(-1 1)"
  })));
};

export default FlagEu;
