// react-country-flags/dist/es/FlagSg.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagSg = function FlagSg(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 4320 2880"
  }, props), React.createElement("path", {
    fill: "#ed2939",
    d: "M0 0h4320v1440H0z"
  }), React.createElement("path", {
    fill: "#fff",
    d: "M0 1440h4320v1440H0zM1481.678 720a541.5 541.5 0 1 1-1083 0 541.5 541.5 0 1 1 1083 0z"
  }), React.createElement("path", {
    fill: "#ed2939",
    d: "M1651.835 720a511.735 511.735 0 1 1-1023.47 0 511.735 511.735 0 1 1 1023.47 0z"
  }), React.createElement("path", {
    fill: "#fff",
    id: "s4a8e7373",
    d: "M1007.195 733.064l-73.56-56.43-73.542 56.457 28.313-90.994-73.795-56.092 91.06.193 27.934-91.123 27.964 91.113 91.06-.226-73.777 56.119 28.343 90.983z"
  }), React.createElement("use", {
    xlinkHref: "#s4a8e7373",
    transform: "translate(577.189)"
  }), React.createElement("use", {
    xlinkHref: "#s4a8e7373",
    transform: "translate(288.889 -214.211)"
  }), React.createElement("use", {
    xlinkHref: "#s4a8e7373",
    transform: "translate(108 342.749)"
  }), React.createElement("use", {
    xlinkHref: "#s4a8e7373",
    transform: "translate(469.189 342.749)"
  }));
};

export default FlagSg;
