// react-country-flags/dist/es/FlagUs.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagUs = function FlagUs(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 7410 3900"
  }, props), React.createElement("path", {
    fill: "#b22234",
    d: "M0 0h7410v3900H0z"
  }), React.createElement("path", {
    d: "M0 450h7410m0 600H0m0 600h7410m0 600H0m0 600h7410m0 600H0",
    stroke: "#fff",
    strokeWidth: 300
  }), React.createElement("path", {
    fill: "#3c3b6e",
    d: "M0 0h2964v2100H0z"
  }), React.createElement("g", {
    fill: "#fff"
  }, React.createElement("g", {
    id: "s363ba562"
  }, React.createElement("g", {
    id: "s0b2b18c6"
  }, React.createElement("g", {
    id: "sa8aa7014"
  }, React.createElement("g", {
    id: "s98f083b9"
  }, React.createElement("path", {
    id: "sd3967715",
    d: "M247 90l70.534 217.082-184.66-134.164h228.253L176.466 307.082z"
  }), React.createElement("use", {
    xlinkHref: "#sd3967715",
    y: 420
  }), React.createElement("use", {
    xlinkHref: "#sd3967715",
    y: 840
  }), React.createElement("use", {
    xlinkHref: "#sd3967715",
    y: 1260
  })), React.createElement("use", {
    xlinkHref: "#sd3967715",
    y: 1680
  })), React.createElement("use", {
    xlinkHref: "#s98f083b9",
    x: 247,
    y: 210
  })), React.createElement("use", {
    xlinkHref: "#s0b2b18c6",
    x: 494
  })), React.createElement("use", {
    xlinkHref: "#s363ba562",
    x: 988
  }), React.createElement("use", {
    xlinkHref: "#s0b2b18c6",
    x: 1976
  }), React.createElement("use", {
    xlinkHref: "#sa8aa7014",
    x: 2470
  })));
};

export default FlagUs;
