// react-country-flags/dist/es/FlagLr.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagLr = function FlagLr(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 209 110"
  }, props), React.createElement("path", {
    fill: "#bf0a30",
    d: "M0 0h209v110H0z"
  }), React.createElement("path", {
    d: "M0 15h209M0 35h209M0 55h209M0 75h209M0 95h209",
    stroke: "#fff",
    strokeWidth: 10
  }), React.createElement("path", {
    fill: "#002868",
    d: "M0 0h50v50H0z"
  }), React.createElement("g", {
    transform: "matrix(15 0 0 15 25 25)"
  }, React.createElement("g", {
    id: "s5ef45845"
  }, React.createElement("path", {
    id: "s677b37c7",
    fill: "#fff",
    transform: "rotate(18 3.157 -.5)",
    d: "M0 0v1h.5z"
  }), React.createElement("use", {
    xlinkHref: "#s677b37c7",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#s5ef45845",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#s5ef45845",
    transform: "rotate(-72)"
  }), React.createElement("use", {
    xlinkHref: "#s5ef45845",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#s5ef45845",
    transform: "rotate(-144)"
  })));
};

export default FlagLr;
