// react-country-flags/dist/es/FlagCv.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagCv = function FlagCv(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 510 300"
  }, props), React.createElement("path", {
    fill: "#003893",
    d: "M0 0h510v300H0z"
  }), React.createElement("path", {
    fill: "#fff",
    d: "M0 150h510v75H0z"
  }), React.createElement("path", {
    fill: "#cf2027",
    d: "M0 175h510v25H0z"
  }), React.createElement("g", {
    fill: "#f7d116",
    transform: "translate(191.25 187.5)"
  }, React.createElement("g", {
    id: "sba450f61"
  }, React.createElement("g", {
    id: "safa48575",
    transform: "translate(0 -75)"
  }, React.createElement("g", {
    id: "s84117a7b"
  }, React.createElement("path", {
    id: "s0088d7a7",
    d: "M0-15V0h7.5",
    transform: "rotate(18 0 -15)"
  }), React.createElement("use", {
    xlinkHref: "#s0088d7a7",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#s84117a7b",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#s84117a7b",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#s84117a7b",
    transform: "rotate(216)"
  }), React.createElement("use", {
    xlinkHref: "#s84117a7b",
    transform: "rotate(288)"
  })), React.createElement("use", {
    xlinkHref: "#safa48575",
    y: 150
  })), React.createElement("use", {
    xlinkHref: "#sba450f61",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#sba450f61",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#sba450f61",
    transform: "rotate(216)"
  }), React.createElement("use", {
    xlinkHref: "#sba450f61",
    transform: "rotate(288)"
  })));
};

export default FlagCv;
