// react-country-flags/dist/es/FlagAn.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagAn = function FlagAn(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "-27 -14 54 36"
  }, props), React.createElement("path", {
    fill: "#fff",
    d: "M-27-14h54v36h-54z"
  }), React.createElement("path", {
    fill: "#dc171d",
    d: "M-6-14H6v36H-6z"
  }), React.createElement("path", {
    fill: "#012a87",
    d: "M-27-2h54v12h-54z"
  }), React.createElement("g", {
    id: "s67c630d5",
    fill: "#fff",
    transform: "scale(1.5)"
  }, React.createElement("g", {
    id: "s9818fb11"
  }, React.createElement("path", {
    id: "s87e52a8c",
    transform: "rotate(18 3.157 -.5)",
    d: "M0 0v1h.5z"
  }), React.createElement("use", {
    xlinkHref: "#s87e52a8c",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#s9818fb11",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#s9818fb11",
    transform: "rotate(-72)"
  }), React.createElement("use", {
    xlinkHref: "#s9818fb11",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#s9818fb11",
    transform: "rotate(-144)"
  })), React.createElement("g", {
    id: "s20030a88"
  }, React.createElement("use", {
    xlinkHref: "#s67c630d5",
    transform: "translate(8 3.314)"
  }), React.createElement("use", {
    xlinkHref: "#s67c630d5",
    transform: "translate(3.314 8)"
  })), React.createElement("use", {
    xlinkHref: "#s20030a88",
    transform: "scale(-1 1)"
  }));
};

export default FlagAn;
