// react-country-flags/dist/es/FlagIn.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagIn = function FlagIn(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 225 150"
  }, props), React.createElement("path", {
    fill: "#f93",
    d: "M0 0h225v150H0z"
  }), React.createElement("path", {
    fill: "#fff",
    d: "M0 50h225v50H0z"
  }), React.createElement("path", {
    fill: "#128807",
    d: "M0 100h225v50H0z"
  }), React.createElement("g", {
    transform: "translate(112.5 75)"
  }, React.createElement("circle", {
    r: 20,
    fill: "#008"
  }), React.createElement("circle", {
    r: 17.5,
    fill: "#fff"
  }), React.createElement("circle", {
    r: 3.5,
    fill: "#008"
  }), React.createElement("g", {
    id: "s802f1c83"
  }, React.createElement("g", {
    id: "sd204b3bc"
  }, React.createElement("g", {
    id: "s19e0389f"
  }, React.createElement("g", {
    id: "s3810a7b3",
    fill: "#008"
  }, React.createElement("circle", {
    r: 0.875,
    transform: "rotate(7.5 -8.75 133.5)"
  }), React.createElement("path", {
    d: "M0 17.5L.6 7 0 2l-.6 5L0 17.5z"
  })), React.createElement("use", {
    xlinkHref: "#s3810a7b3",
    transform: "rotate(15)"
  })), React.createElement("use", {
    xlinkHref: "#s19e0389f",
    transform: "rotate(30)"
  })), React.createElement("use", {
    xlinkHref: "#sd204b3bc",
    transform: "rotate(60)"
  })), React.createElement("use", {
    xlinkHref: "#s802f1c83",
    transform: "rotate(120)"
  }), React.createElement("use", {
    xlinkHref: "#s802f1c83",
    transform: "rotate(-120)"
  })));
};

export default FlagIn;
