// react-country-flags/dist/es/FlagSo.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagSo = function FlagSo(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 81 54"
  }, props), React.createElement("path", {
    fill: "#4189DD",
    d: "M0 0h81v54H0z"
  }), React.createElement("g", {
    transform: "matrix(13 0 0 13 40.5 27)"
  }, React.createElement("g", {
    id: "seea529c4"
  }, React.createElement("path", {
    id: "s529b48d0",
    fill: "#FFF",
    transform: "rotate(18 3.157 -.5)",
    d: "M0 0v1h.5z"
  }), React.createElement("use", {
    xlinkHref: "#s529b48d0",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#seea529c4",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#seea529c4",
    transform: "rotate(-72)"
  }), React.createElement("use", {
    xlinkHref: "#seea529c4",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#seea529c4",
    transform: "rotate(-144)"
  })));
};

export default FlagSo;
