// react-country-flags/dist/es/FlagKp.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagKp = function FlagKp(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 72 36"
  }, props), React.createElement("path", {
    fill: "#024FA2",
    d: "M0 0h72v36H0z"
  }), React.createElement("path", {
    fill: "#fff",
    d: "M0 6h72v24H0z"
  }), React.createElement("path", {
    fill: "#ED1C27",
    d: "M0 7h72v22H0z"
  }), React.createElement("circle", {
    fill: "#fff",
    cx: 24,
    cy: 18,
    r: 8
  }), React.createElement("g", {
    transform: "matrix(7.75 0 0 7.75 24 18)",
    fill: "#ED1C27"
  }, React.createElement("g", {
    id: "s0f34304b"
  }, React.createElement("path", {
    id: "s535ea555",
    transform: "rotate(18 3.157 -.5)",
    d: "M0 0v1h.5z"
  }), React.createElement("use", {
    xlinkHref: "#s535ea555",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#s0f34304b",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#s0f34304b",
    transform: "rotate(-72)"
  }), React.createElement("use", {
    xlinkHref: "#s0f34304b",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#s0f34304b",
    transform: "rotate(-144)"
  })));
};

export default FlagKp;
