// react-country-flags/dist/es/FlagGe.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagGe = function FlagGe(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 300 200"
  }, props), React.createElement("defs", null, React.createElement("g", {
    id: "s04dc8ef0"
  }, React.createElement("clipPath", {
    id: "s28a96dfc"
  }, React.createElement("path", {
    d: "M-109 104a104 104 0 0 0 0-208h218a104 104 0 0 0 0 208z"
  })), React.createElement("path", {
    id: "s68a700b4",
    d: "M-55 74a55 55 0 0 1 110 0V-74a55 55 0 0 1-110 0z",
    clipPath: "url(#s28a96dfc)"
  }), React.createElement("use", {
    xlinkHref: "#s68a700b4",
    transform: "rotate(90)"
  }))), React.createElement("path", {
    fill: "#fff",
    d: "M0 0h300v200H0z"
  }), React.createElement("path", {
    d: "M130 0v80H0v40h130v80h40v-80h130V80H170V0h-40z",
    fill: "red"
  }), React.createElement("use", {
    xlinkHref: "#s04dc8ef0",
    transform: "translate(64.45 39.45)",
    fill: "red"
  }), React.createElement("use", {
    xlinkHref: "#s04dc8ef0",
    transform: "translate(235.55 160.55)",
    fill: "red"
  }), React.createElement("use", {
    xlinkHref: "#s04dc8ef0",
    transform: "translate(235.55 39.45)",
    fill: "red"
  }), React.createElement("use", {
    xlinkHref: "#s04dc8ef0",
    transform: "translate(64.45 160.55)",
    fill: "red"
  }));
};

export default FlagGe;
