// react-country-flags/dist/es/FlagCn.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagCn = function FlagCn(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 30 20"
  }, props), React.createElement("defs", null, React.createElement("path", {
    id: "sd8bd4f91",
    d: "M0-1L.588.809-.952-.309H.952L-.588.809z",
    fill: "#ffde00"
  })), React.createElement("path", {
    fill: "#de2910",
    d: "M0 0h30v20H0z"
  }), React.createElement("use", {
    xlinkHref: "#sd8bd4f91",
    transform: "matrix(3 0 0 3 5 5)"
  }), React.createElement("use", {
    xlinkHref: "#sd8bd4f91",
    transform: "rotate(23.036 .093 25.536)"
  }), React.createElement("use", {
    xlinkHref: "#sd8bd4f91",
    transform: "rotate(45.87 1.273 16.18)"
  }), React.createElement("use", {
    xlinkHref: "#sd8bd4f91",
    transform: "rotate(69.945 .996 12.078)"
  }), React.createElement("use", {
    xlinkHref: "#sd8bd4f91",
    transform: "rotate(20.66 -19.689 31.932)"
  }));
};

export default FlagCn;
