// react-country-flags/dist/es/FlagTk.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagTk = function FlagTk(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 1800 900"
  }, props), React.createElement("defs", null, React.createElement("g", {
    id: "sfdf72aaf"
  }, React.createElement("g", {
    id: "s44464cf3"
  }, React.createElement("path", {
    d: "M1 0H0v.5z",
    transform: "translate(0 -.325)"
  }), React.createElement("path", {
    d: "M1 0H0v-.5z",
    transform: "rotate(-36 .5 -.162)"
  })), React.createElement("use", {
    xlinkHref: "#s44464cf3",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#s44464cf3",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#s44464cf3",
    transform: "rotate(216)"
  }), React.createElement("use", {
    xlinkHref: "#s44464cf3",
    transform: "rotate(288)"
  }))), React.createElement("path", {
    fill: "#00247d",
    d: "M0 0h1800v900H0z"
  }), React.createElement("g", {
    fill: "#fed100"
  }, React.createElement("path", {
    d: "M329.376 777.265c-17.934-.15 168.624-99.431 324.77-226.6C858.788 384 1299.54 117.157 1429.115 73.796c13.543-4.532-27.776 23.073-33.14 32.016-138.341 163.74-27.925 472.724 144.973 628.048 51.99 39.763 49.327 41.85 145.707 45.746v9l-1357.28-11.34zM318.125 795.325S304.962 804.727 305 811.73c.043 7.772 14.766 18.047 14.766 18.047l1338.745 14.765 24.609-16.406-34.453-21.328-1330.542-11.484z"
  })), React.createElement("g", {
    fill: "#fff"
  }, React.createElement("use", {
    xlinkHref: "#sfdf72aaf",
    transform: "matrix(45 0 0 45 342.957 112.581)"
  }), React.createElement("use", {
    xlinkHref: "#sfdf72aaf",
    transform: "matrix(37.5 0 0 37.5 521.205 267.53)"
  }), React.createElement("use", {
    xlinkHref: "#sfdf72aaf",
    transform: "matrix(45 0 0 45 135 321.808)"
  }), React.createElement("use", {
    xlinkHref: "#sfdf72aaf",
    transform: "matrix(52.5 0 0 52.5 342.957 652.581)"
  })));
};

export default FlagTk;
