// react-country-flags/dist/es/FlagPh.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagPh = function FlagPh(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 180 90"
  }, props), React.createElement("path", {
    fill: "#0038a8",
    d: "M0 0h180v90H0z"
  }), React.createElement("path", {
    fill: "#ce1126",
    d: "M0 45h180v45H0z"
  }), React.createElement("path", {
    d: "M77.942 45L0 90V0",
    fill: "#fff"
  }), React.createElement("g", {
    transform: "translate(28 45)",
    fill: "#fcd116"
  }, React.createElement("circle", {
    r: 9
  }), React.createElement("g", {
    id: "s94ec592d"
  }, React.createElement("g", {
    id: "sf31cda82"
  }, React.createElement("g", {
    id: "sa25ec7ea"
  }, React.createElement("path", {
    d: "M-1 0l.062.062L0 0l-.938-.062z",
    transform: "scale(19)"
  }), React.createElement("path", {
    id: "s7c77c517",
    d: "M-.884.116l.05.05L0 0z",
    transform: "scale(19.2381)"
  }), React.createElement("use", {
    xlinkHref: "#s7c77c517",
    transform: "scale(1 -1)"
  })), React.createElement("use", {
    xlinkHref: "#sa25ec7ea",
    transform: "rotate(45)"
  })), React.createElement("use", {
    xlinkHref: "#sf31cda82",
    transform: "rotate(90)"
  })), React.createElement("use", {
    xlinkHref: "#s94ec592d",
    transform: "rotate(180)"
  }), React.createElement("g", {
    transform: "translate(-2.02)"
  }, React.createElement("g", {
    id: "s9adb718d",
    transform: "translate(37.962)"
  }, React.createElement("path", {
    id: "sfefc3cc8",
    d: "M5 0L1.618 1.176l-.073 3.58-2.163-2.854-3.427 1.037L-2 0z"
  }), React.createElement("use", {
    xlinkHref: "#sfefc3cc8",
    transform: "scale(1 -1)"
  })), React.createElement("use", {
    xlinkHref: "#s9adb718d",
    transform: "rotate(120)"
  }), React.createElement("use", {
    xlinkHref: "#s9adb718d",
    transform: "rotate(-120)"
  }))));
};

export default FlagPh;
