// react-country-flags/dist/es/FlagNp.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagNp = function FlagNp(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "-17.582 -4.664 71.571 87.246"
  }, props), React.createElement("use", {
    xlinkHref: "#sab34580f",
    stroke: "#003893",
    strokeWidth: 5.165
  }), React.createElement("path", {
    id: "sab34580f",
    d: "M-15 37.574h60L-15 0v80h60l-60-60z",
    fill: "#DC143C"
  }), React.createElement("g", {
    fill: "#fff"
  }, React.createElement("path", {
    d: "M-11.95 23.483a12.84 12.84 0 0 0 23.9 0 11.95 11.95 0 0 1-23.9 0"
  }), React.createElement("g", {
    transform: "translate(0 29.045) scale(5.56106)"
  }, React.createElement("circle", {
    r: 1
  }), React.createElement("g", {
    id: "sdf2c85fb"
  }, React.createElement("g", {
    id: "se6643760"
  }, React.createElement("path", {
    id: "see3d2fc5",
    d: "M.195-.98L0-1.39l-.195.408",
    transform: "rotate(11.25)"
  }), React.createElement("use", {
    xlinkHref: "#see3d2fc5",
    transform: "rotate(22.5)"
  }), React.createElement("use", {
    xlinkHref: "#see3d2fc5",
    transform: "rotate(45)"
  })), React.createElement("use", {
    xlinkHref: "#se6643760",
    transform: "rotate(67.5)"
  })), React.createElement("use", {
    xlinkHref: "#sdf2c85fb",
    transform: "scale(-1 1)"
  })), React.createElement("g", {
    transform: "matrix(8.1434 0 0 8.1434 0 58.787)"
  }, React.createElement("circle", {
    r: 1
  }), React.createElement("g", {
    id: "s1830dce3"
  }, React.createElement("g", {
    id: "s3d3b3ac3"
  }, React.createElement("path", {
    id: "s36724cc8",
    d: "M.259.966L0 1.576l-.259-.61"
  }), React.createElement("use", {
    xlinkHref: "#s36724cc8",
    transform: "rotate(180)"
  })), React.createElement("use", {
    xlinkHref: "#s3d3b3ac3",
    transform: "rotate(90)"
  })), React.createElement("use", {
    xlinkHref: "#s1830dce3",
    transform: "rotate(30)"
  }), React.createElement("use", {
    xlinkHref: "#s1830dce3",
    transform: "rotate(60)"
  }))));
};

export default FlagNp;
