// react-country-flags/dist/es/FlagKr.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagKr = function FlagKr(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "-36 -24 72 48"
  }, props), React.createElement("path", {
    fill: "#fff",
    d: "M-36-24h72v48h-72z"
  }), React.createElement("g", {
    transform: "rotate(-56.31)"
  }, React.createElement("g", {
    id: "s3f8924d6"
  }, React.createElement("path", {
    id: "sf14b4394",
    d: "M-6-25H6m-12 3H6m-12 3H6",
    stroke: "#000",
    strokeWidth: 2
  }), React.createElement("use", {
    xlinkHref: "#sf14b4394",
    y: 44
  })), React.createElement("path", {
    stroke: "#fff",
    d: "M0 17v10"
  }), React.createElement("circle", {
    fill: "#c60c30",
    r: 12
  }), React.createElement("path", {
    fill: "#003478",
    d: "M0-12A6 6 0 0 0 0 0a6 6 0 0 1 0 12 12 12 0 0 1 0-24z"
  })), React.createElement("g", {
    transform: "rotate(-123.69)"
  }, React.createElement("use", {
    xlinkHref: "#s3f8924d6"
  }), React.createElement("path", {
    stroke: "#fff",
    d: "M0-23.5v3M0 17v3.5m0 3v3"
  })));
};

export default FlagKr;
