// react-country-flags/dist/es/FlagPa.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagPa = function FlagPa(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 36 24"
  }, props), React.createElement("path", {
    fill: "#fff",
    d: "M0 0h36v24H0z"
  }), React.createElement("g", {
    fill: "#005293"
  }, React.createElement("g", {
    id: "s95443cde",
    transform: "translate(9 6)"
  }, React.createElement("g", {
    id: "s22632413"
  }, React.createElement("path", {
    id: "s94eef583",
    d: "M0-3v3h1.5z",
    transform: "rotate(18 0 -3)"
  }), React.createElement("use", {
    xlinkHref: "#s94eef583",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#s22632413",
    transform: "rotate(72)"
  }), React.createElement("use", {
    xlinkHref: "#s22632413",
    transform: "rotate(144)"
  }), React.createElement("use", {
    xlinkHref: "#s22632413",
    transform: "rotate(216)"
  }), React.createElement("use", {
    xlinkHref: "#s22632413",
    transform: "rotate(288)"
  })), React.createElement("path", {
    d: "M0 12h18v12H0z"
  })), React.createElement("g", {
    fill: "#d21034"
  }, React.createElement("path", {
    d: "M18 0h18v12H18z"
  }), React.createElement("use", {
    xlinkHref: "#s95443cde",
    x: 18,
    y: 12
  })));
};

export default FlagPa;
