// react-country-flags/dist/es/FlagKe.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagKe = function FlagKe(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "-120 -80 240 160"
  }, props), React.createElement("defs", null, React.createElement("path", {
    id: "s87a071ba",
    d: "M-1 55.426h2V-38c2-2 2-5 2-8 0-2 0-10-3-18.663C-3-56-3-48-3-46c0 3 0 6 2 8z",
    strokeMiterlimit: 10,
    transform: "rotate(30)"
  })), React.createElement("path", {
    fill: "#fff",
    d: "M-120-80h240V80h-240z"
  }), React.createElement("path", {
    d: "M-120-80h240v48h-240z"
  }), React.createElement("path", {
    fill: "#060",
    d: "M-120 32h240v48h-240z"
  }), React.createElement("g", {
    id: "s70d6d5e8"
  }, React.createElement("use", {
    xlinkHref: "#s87a071ba",
    stroke: "#000"
  }), React.createElement("use", {
    xlinkHref: "#s87a071ba",
    fill: "#fff"
  })), React.createElement("use", {
    xlinkHref: "#s70d6d5e8",
    transform: "scale(-1 1)"
  }), React.createElement("path", {
    fill: "#b00",
    d: "M-120-24v48h101c3 8 13 24 19 24s16-16 19-24h101v-48H19C16-32 6-48 0-48s-16 16-19 24z"
  }), React.createElement("path", {
    id: "s8db5068e",
    d: "M19 24c3-8 5-16 5-24s-2-16-5-24c-3 8-5 16-5 24s2 16 5 24"
  }), React.createElement("use", {
    xlinkHref: "#s8db5068e",
    transform: "scale(-1 1)"
  }), React.createElement("g", {
    fill: "#fff"
  }, React.createElement("ellipse", {
    rx: 4,
    ry: 6
  }), React.createElement("path", {
    id: "s9ec57b4b",
    d: "M1 5.85s4 8 4 21-4 21-4 21z"
  }), React.createElement("use", {
    xlinkHref: "#s9ec57b4b",
    transform: "scale(-1)"
  }), React.createElement("use", {
    xlinkHref: "#s9ec57b4b",
    transform: "scale(-1 1)"
  }), React.createElement("use", {
    xlinkHref: "#s9ec57b4b",
    transform: "scale(1 -1)"
  })));
};

export default FlagKe;
