// react-country-flags/dist/es/FlagTl.js
import { a as _extends } from './chunk-62fb3b11.js';
import React from 'react';

var FlagTl = function FlagTl(props) {
  return React.createElement("svg", _extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "-3 -6 24 12"
  }, props), React.createElement("path", {
    d: "M21 6V-6H-3V6z",
    fill: "#dc241f"
  }), React.createElement("path", {
    d: "M-3-6V6L9 0z",
    fill: "#ffc726"
  }), React.createElement("path", {
    d: "M-3-6V6l8-6z"
  }), React.createElement("g", {
    transform: "rotate(-26.565)"
  }, React.createElement("g", {
    id: "s21cd84c8"
  }, React.createElement("g", {
    id: "s74890357"
  }, React.createElement("path", {
    d: "M0-2.1V0h1z",
    fill: "#fff",
    transform: "rotate(18 0 -2.1)",
    id: "s152cec9a"
  }), React.createElement("use", {
    xlinkHref: "#s152cec9a",
    transform: "scale(-1 1)"
  })), React.createElement("use", {
    xlinkHref: "#s74890357",
    transform: "rotate(72)"
  })), React.createElement("use", {
    xlinkHref: "#s74890357",
    transform: "rotate(-72)"
  }), React.createElement("use", {
    xlinkHref: "#s21cd84c8",
    transform: "rotate(144)"
  })));
};

export default FlagTl;
