// react-country-flags/dist/cjs/FlagLy.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagLy = function FlagLy(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    viewBox: "0 0 960 480"
  }, props), React__default.createElement("path", {
    fill: "#239e46",
    d: "M0 0h960v480H0z"
  }), React__default.createElement("path", {
    d: "M0 0h960v360H0z"
  }), React__default.createElement("path", {
    fill: "#e70013",
    d: "M0 0h960v120H0z"
  }), React__default.createElement("circle", {
    cx: 480,
    cy: 240,
    r: 60,
    fill: "#fff"
  }), React__default.createElement("circle", {
    cx: 492.132,
    cy: 240,
    r: 52.132
  }), React__default.createElement("path", {
    d: "M509.175 240l80.7-26.221L540 282.426v-84.852l49.875 68.647z",
    fill: "#fff"
  }));
};

module.exports = FlagLy;
