// react-country-flags/dist/cjs/FlagGb.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagGb = function FlagGb(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    viewBox: "0 0 60 30"
  }, props), React__default.createElement("clipPath", {
    id: "a"
  }, React__default.createElement("path", {
    d: "M30 15h30v15zv15H0zH0V0zV0h30z"
  })), React__default.createElement("path", {
    d: "M0 0v30h60V0z",
    fill: "#00247d"
  }), React__default.createElement("path", {
    d: "M0 0l60 30m0-30L0 30",
    stroke: "#fff",
    strokeWidth: 6
  }), React__default.createElement("path", {
    d: "M0 0l60 30m0-30L0 30",
    clipPath: "url(#a)",
    stroke: "#cf142b",
    strokeWidth: 4
  }), React__default.createElement("path", {
    d: "M30 0v30M0 15h60",
    stroke: "#fff",
    strokeWidth: 10
  }), React__default.createElement("path", {
    d: "M30 0v30M0 15h60",
    stroke: "#cf142b",
    strokeWidth: 6
  }));
};

module.exports = FlagGb;
