// react-country-flags/dist/cjs/FlagVe.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagVe = function FlagVe(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 180 120"
  }, props), React__default.createElement("defs", null, React__default.createElement("g", {
    id: "s17194599",
    transform: "translate(0 -36)"
  }, React__default.createElement("g", {
    id: "sb6cb2187"
  }, React__default.createElement("g", {
    id: "s70610266"
  }, React__default.createElement("path", {
    d: "M0-5v5h3z",
    fill: "#fff",
    transform: "rotate(18 0 -5)",
    id: "sa8143e40"
  }), React__default.createElement("use", {
    xlinkHref: "#sa8143e40",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s70610266",
    transform: "rotate(72)"
  })), React__default.createElement("use", {
    xlinkHref: "#s70610266",
    transform: "rotate(-72)"
  }), React__default.createElement("use", {
    xlinkHref: "#sb6cb2187",
    transform: "rotate(144)"
  }))), React__default.createElement("path", {
    d: "M0 0h180v120H0z",
    fill: "#cf142b"
  }), React__default.createElement("path", {
    d: "M0 0h180v80H0z",
    fill: "#00247d"
  }), React__default.createElement("path", {
    d: "M0 0h180v40H0z",
    fill: "#fc0"
  }), React__default.createElement("g", {
    transform: "translate(90 84)"
  }, React__default.createElement("g", {
    id: "s5e5470d0"
  }, React__default.createElement("g", {
    id: "sf6865a85"
  }, React__default.createElement("use", {
    xlinkHref: "#s17194599",
    transform: "rotate(10)"
  }), React__default.createElement("use", {
    xlinkHref: "#s17194599",
    transform: "rotate(30)"
  })), React__default.createElement("use", {
    xlinkHref: "#sf6865a85",
    transform: "rotate(40)"
  })), React__default.createElement("use", {
    xlinkHref: "#s5e5470d0",
    transform: "rotate(-80)"
  })));
};

module.exports = FlagVe;
