// react-country-flags/dist/cjs/FlagAz.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagAz = function FlagAz(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    viewBox: "0 0 1200 600"
  }, props), React__default.createElement("path", {
    fill: "#3f9c35",
    d: "M0 0h1200v600H0z"
  }), React__default.createElement("path", {
    fill: "#ed2939",
    d: "M0 0h1200v400H0z"
  }), React__default.createElement("path", {
    fill: "#00b9e4",
    d: "M0 0h1200v200H0z"
  }), React__default.createElement("circle", {
    cx: 580,
    cy: 300,
    r: 90,
    fill: "#fff"
  }), React__default.createElement("circle", {
    cx: 600,
    cy: 300,
    r: 75,
    fill: "#ed2939"
  }), React__default.createElement("path", {
    d: "M680 250l9.567 26.903 25.788-12.258-12.258 25.788L730 300l-26.903 9.567 12.258 25.788-25.788-12.258L680 350l-9.567-26.903-25.788 12.258 12.258-25.788L630 300l26.903-9.567-12.258-25.788 25.788 12.258L680 250z",
    fill: "#fff"
  }));
};

module.exports = FlagAz;
