// react-country-flags/dist/cjs/FlagPr.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagPr = function FlagPr(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 45 30"
  }, props), React__default.createElement("path", {
    fill: "#ED0000",
    d: "M0 0h45v30H0z"
  }), React__default.createElement("path", {
    stroke: "#FFF",
    strokeWidth: 6,
    d: "M0 9h45M0 21h45"
  }), React__default.createElement("path", {
    fill: "#0050F0",
    d: "M0 0l25.98 15L0 30z"
  }), React__default.createElement("g", {
    fill: "#FFF",
    transform: "matrix(5 0 0 5 8.66 15)"
  }, React__default.createElement("g", {
    id: "s53d7d484"
  }, React__default.createElement("path", {
    id: "s80f072d4",
    transform: "rotate(18 3.157 -.5)",
    d: "M0 0v1h.5z"
  }), React__default.createElement("use", {
    xlinkHref: "#s80f072d4",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s53d7d484",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s53d7d484",
    transform: "rotate(-72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s53d7d484",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#s53d7d484",
    transform: "rotate(-144)"
  })));
};

module.exports = FlagPr;
