// react-country-flags/dist/cjs/FlagCk.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagCk = function FlagCk(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 600 300"
  }, props), React__default.createElement("clipPath", {
    id: "s85203e2b"
  }, React__default.createElement("path", {
    d: "M0 0v75h350v75h-50zm300 0H150v200H0v-50z"
  })), React__default.createElement("path", {
    fill: "#00247d",
    d: "M0 0h600v300H0z"
  }), React__default.createElement("path", {
    d: "M0 0l300 150m0-150L0 150",
    stroke: "#fff",
    strokeWidth: 30
  }), React__default.createElement("path", {
    d: "M0 0l300 150m0-150L0 150",
    stroke: "#cf142b",
    strokeWidth: 20,
    clipPath: "url(#s85203e2b)"
  }), React__default.createElement("path", {
    d: "M150 0v200M0 75h350",
    stroke: "#fff",
    strokeWidth: 50
  }), React__default.createElement("path", {
    d: "M150 0v200M0 75h350",
    stroke: "#cf142b",
    strokeWidth: 30
  }), React__default.createElement("path", {
    d: "M300 0v150H0v150h600V0z",
    fill: "#00247d"
  }), React__default.createElement("g", {
    transform: "translate(450 150)"
  }, React__default.createElement("g", {
    id: "s49ffa104",
    transform: "translate(0 -102)",
    fill: "#fff"
  }, React__default.createElement("g", {
    id: "s39779b07"
  }, React__default.createElement("path", {
    id: "sc43daf61",
    d: "M0-22.5V0h12",
    transform: "rotate(18 0 -22.5)"
  }), React__default.createElement("use", {
    xlinkHref: "#sc43daf61",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s39779b07",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s39779b07",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#s39779b07",
    transform: "rotate(216)"
  }), React__default.createElement("use", {
    xlinkHref: "#s39779b07",
    transform: "rotate(288)"
  })), React__default.createElement("g", {
    id: "s78382fbe"
  }, React__default.createElement("use", {
    xlinkHref: "#s49ffa104",
    transform: "rotate(24)"
  }), React__default.createElement("use", {
    xlinkHref: "#s49ffa104",
    transform: "rotate(48)"
  }), React__default.createElement("use", {
    xlinkHref: "#s49ffa104",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s49ffa104",
    transform: "rotate(96)"
  }), React__default.createElement("use", {
    xlinkHref: "#s49ffa104",
    transform: "rotate(120)"
  }), React__default.createElement("use", {
    xlinkHref: "#s49ffa104",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#s49ffa104",
    transform: "rotate(168)"
  })), React__default.createElement("use", {
    xlinkHref: "#s78382fbe",
    transform: "rotate(168)"
  })));
};

module.exports = FlagCk;
