// react-country-flags/dist/cjs/FlagIl.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagIl = function FlagIl(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 220 160"
  }, props), React__default.createElement("defs", null, React__default.createElement("path", {
    id: "s9038471d",
    stroke: "#0038b8",
    strokeWidth: 5.5,
    fillOpacity: 0,
    d: "M0-29.141l-25.237 43.712h50.474z"
  })), React__default.createElement("path", {
    fill: "#fff",
    d: "M0 0h220v160H0z"
  }), React__default.createElement("path", {
    fill: "#0038b8",
    d: "M0 15h220v25H0zM0 120h220v25H0z"
  }), React__default.createElement("use", {
    xlinkHref: "#s9038471d",
    transform: "translate(110 80)"
  }), React__default.createElement("use", {
    xlinkHref: "#s9038471d",
    transform: "rotate(180 55 40)"
  }));
};

module.exports = FlagIl;
