// react-country-flags/dist/cjs/FlagEt.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagEt = function FlagEt(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 720 360"
  }, props), React__default.createElement("path", {
    fill: "#da121a",
    d: "M0 0h720v360H0z"
  }), React__default.createElement("path", {
    fill: "#fcdd09",
    d: "M0 0h720v240H0z"
  }), React__default.createElement("path", {
    fill: "#078930",
    d: "M0 0h720v120H0z"
  }), React__default.createElement("g", {
    transform: "translate(360 180)"
  }, React__default.createElement("circle", {
    fill: "#0f47af",
    r: 120
  }), React__default.createElement("g", {
    id: "se519047c"
  }, React__default.createElement("path", {
    fill: "#fcdd09",
    d: "M0-96l-4.206 12.944 17.348 53.39h-23.13l-2.599 8h74.163l11.011-8H21.553z"
  }), React__default.createElement("path", {
    stroke: "#fcdd09",
    strokeWidth: 4,
    d: "M25.863-35.597l30.564-42.069"
  })), React__default.createElement("use", {
    xlinkHref: "#se519047c",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#se519047c",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#se519047c",
    transform: "rotate(216)"
  }), React__default.createElement("use", {
    xlinkHref: "#se519047c",
    transform: "rotate(288)"
  })));
};

module.exports = FlagEt;
