// react-country-flags/dist/cjs/FlagMh.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagMh = function FlagMh(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 570 300"
  }, props), React__default.createElement("path", {
    fill: "#003893",
    d: "M0 0h570v300H0z"
  }), React__default.createElement("path", {
    d: "M0 295.2V288L570 35v87.4z",
    fill: "#fff"
  }), React__default.createElement("path", {
    d: "M0 290.4v-4.8L570 4.8v58.8z",
    fill: "#dd7500"
  }), React__default.createElement("g", {
    fill: "#fff",
    transform: "translate(109.536 109.536)"
  }, React__default.createElement("g", {
    id: "s082e676b"
  }, React__default.createElement("path", {
    d: "M0-93L5.59 0H-5.59z"
  }), React__default.createElement("path", {
    id: "se1b5eca9",
    d: "M0-66.588L6.824 0H-6.824z",
    transform: "rotate(15)"
  }), React__default.createElement("use", {
    xlinkHref: "#se1b5eca9",
    transform: "rotate(15)"
  }), React__default.createElement("use", {
    xlinkHref: "#se1b5eca9",
    transform: "rotate(30)"
  }), React__default.createElement("use", {
    xlinkHref: "#se1b5eca9",
    transform: "rotate(45)"
  }), React__default.createElement("use", {
    xlinkHref: "#se1b5eca9",
    transform: "rotate(60)"
  })), React__default.createElement("use", {
    xlinkHref: "#s082e676b",
    transform: "rotate(90)"
  }), React__default.createElement("use", {
    xlinkHref: "#s082e676b",
    transform: "rotate(180)"
  }), React__default.createElement("use", {
    xlinkHref: "#s082e676b",
    transform: "rotate(270)"
  })));
};

module.exports = FlagMh;
