// react-country-flags/dist/cjs/FlagEu.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagEu = function FlagEu(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 810 540"
  }, props), React__default.createElement("defs", null, React__default.createElement("g", {
    id: "s91bf8332"
  }, React__default.createElement("g", {
    id: "sf412d9e4"
  }, React__default.createElement("path", {
    id: "s0f53adc6",
    d: "M0 0v1h.5z",
    transform: "rotate(18 3.157 -.5)"
  }), React__default.createElement("use", {
    xlinkHref: "#s0f53adc6",
    transform: "scale(-1 1)"
  })), React__default.createElement("g", {
    id: "sa8efbd1a"
  }, React__default.createElement("use", {
    xlinkHref: "#sf412d9e4",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#sf412d9e4",
    transform: "rotate(144)"
  })), React__default.createElement("use", {
    xlinkHref: "#sa8efbd1a",
    transform: "scale(-1 1)"
  }))), React__default.createElement("path", {
    fill: "#039",
    d: "M0 0h810v540H0z"
  }), React__default.createElement("g", {
    fill: "#fc0",
    transform: "matrix(30 0 0 30 405 270)"
  }, React__default.createElement("use", {
    xlinkHref: "#s91bf8332",
    y: -6
  }), React__default.createElement("use", {
    xlinkHref: "#s91bf8332",
    y: 6
  }), React__default.createElement("g", {
    id: "s5cea22cb"
  }, React__default.createElement("use", {
    xlinkHref: "#s91bf8332",
    x: -6
  }), React__default.createElement("use", {
    xlinkHref: "#s91bf8332",
    transform: "rotate(-144 -2.344 -2.11)"
  }), React__default.createElement("use", {
    xlinkHref: "#s91bf8332",
    transform: "rotate(144 -2.11 -2.344)"
  }), React__default.createElement("use", {
    xlinkHref: "#s91bf8332",
    transform: "rotate(72 -4.663 -2.076)"
  }), React__default.createElement("use", {
    xlinkHref: "#s91bf8332",
    transform: "rotate(72 -5.076 .534)"
  })), React__default.createElement("use", {
    xlinkHref: "#s5cea22cb",
    transform: "scale(-1 1)"
  })));
};

module.exports = FlagEu;
