// react-country-flags/dist/cjs/FlagCv.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagCv = function FlagCv(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 510 300"
  }, props), React__default.createElement("path", {
    fill: "#003893",
    d: "M0 0h510v300H0z"
  }), React__default.createElement("path", {
    fill: "#fff",
    d: "M0 150h510v75H0z"
  }), React__default.createElement("path", {
    fill: "#cf2027",
    d: "M0 175h510v25H0z"
  }), React__default.createElement("g", {
    fill: "#f7d116",
    transform: "translate(191.25 187.5)"
  }, React__default.createElement("g", {
    id: "sba450f61"
  }, React__default.createElement("g", {
    id: "safa48575",
    transform: "translate(0 -75)"
  }, React__default.createElement("g", {
    id: "s84117a7b"
  }, React__default.createElement("path", {
    id: "s0088d7a7",
    d: "M0-15V0h7.5",
    transform: "rotate(18 0 -15)"
  }), React__default.createElement("use", {
    xlinkHref: "#s0088d7a7",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s84117a7b",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s84117a7b",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#s84117a7b",
    transform: "rotate(216)"
  }), React__default.createElement("use", {
    xlinkHref: "#s84117a7b",
    transform: "rotate(288)"
  })), React__default.createElement("use", {
    xlinkHref: "#safa48575",
    y: 150
  })), React__default.createElement("use", {
    xlinkHref: "#sba450f61",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#sba450f61",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#sba450f61",
    transform: "rotate(216)"
  }), React__default.createElement("use", {
    xlinkHref: "#sba450f61",
    transform: "rotate(288)"
  })));
};

module.exports = FlagCv;
