// react-country-flags/dist/cjs/FlagGd.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagGd = function FlagGd(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 500 300"
  }, props), React__default.createElement("defs", null, React__default.createElement("g", {
    id: "s6d4f0aa3"
  }, React__default.createElement("g", {
    id: "s166eb571"
  }, React__default.createElement("path", {
    id: "sad002c60",
    d: "M0-1v1h.5",
    transform: "rotate(18 0 -1)",
    fill: "#fcd116"
  }), React__default.createElement("use", {
    xlinkHref: "#sad002c60",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s166eb571",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s166eb571",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#s166eb571",
    transform: "rotate(216)"
  }), React__default.createElement("use", {
    xlinkHref: "#s166eb571",
    transform: "rotate(288)"
  }))), React__default.createElement("path", {
    fill: "#ce1126",
    d: "M0 0h500v300H0z"
  }), React__default.createElement("path", {
    fill: "#007a5e",
    d: "M42 42h416v216H42z"
  }), React__default.createElement("path", {
    d: "M42 42h416L42 258h416z",
    fill: "#fcd116"
  }), React__default.createElement("circle", {
    cx: 250,
    cy: 150,
    r: 36,
    fill: "#ce1126"
  }), React__default.createElement("use", {
    xlinkHref: "#s6d4f0aa3",
    transform: "matrix(33 0 0 33 250 150)"
  }), React__default.createElement("use", {
    xlinkHref: "#s5519e543",
    x: -100
  }), React__default.createElement("use", {
    id: "s5519e543",
    xlinkHref: "#s6d4f0aa3",
    transform: "matrix(19.5 0 0 19.5 250 21)"
  }), React__default.createElement("use", {
    xlinkHref: "#s5519e543",
    x: 100
  }), React__default.createElement("use", {
    xlinkHref: "#s5519e543",
    x: -100,
    y: 258
  }), React__default.createElement("use", {
    xlinkHref: "#s5519e543",
    y: 258
  }), React__default.createElement("use", {
    xlinkHref: "#s5519e543",
    x: 100,
    y: 258
  }), React__default.createElement("path", {
    d: "M67.749 150.41c4.504 8.39 13.265 17.52 20.916 20.73.123-8.52-2.9-19.44-7.034-28.14l-13.882 7.41z",
    fill: "#ce1126"
  }), React__default.createElement("path", {
    d: "M60.112 121.63c6.529 13.61-16.933 46.08 22.156 53.69-4.822-6.58-7.931-17.44-6.755-26.16 8.201 3.12 16.83 12.25 20.317 19.23 10.23-37.15-26.24-34.89-35.718-46.76z",
    fill: "#fcd116"
  }));
};

module.exports = FlagGd;
