// react-country-flags/dist/cjs/FlagNz.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagNz = function FlagNz(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 1200 600"
  }, props), React__default.createElement("defs", null, React__default.createElement("clipPath", {
    id: "sa3487802"
  }, React__default.createElement("path", {
    d: "M0 0h600v300H0z"
  })), React__default.createElement("clipPath", {
    id: "sa841c70d"
  }, React__default.createElement("path", {
    d: "M0 0l300 150H0zm300 0h300L300 150zm0 150h300v150zm0 0v150H0z"
  })), React__default.createElement("g", {
    id: "s4ed22915"
  }, React__default.createElement("g", {
    id: "s7f67afd8"
  }, React__default.createElement("path", {
    d: "M0 0v.5L1 0z",
    transform: "translate(0 -.325)"
  }), React__default.createElement("path", {
    d: "M0 0v-.5L1 0z",
    transform: "rotate(-36 .5 -.162)"
  })), React__default.createElement("use", {
    xlinkHref: "#s7f67afd8",
    transform: "scale(-1 1)"
  }), React__default.createElement("use", {
    xlinkHref: "#s7f67afd8",
    transform: "rotate(72 0 0)"
  }), React__default.createElement("use", {
    xlinkHref: "#s7f67afd8",
    transform: "rotate(-72 0 0)"
  }), React__default.createElement("use", {
    xlinkHref: "#s7f67afd8",
    transform: "scale(-1 1) rotate(72)"
  }))), React__default.createElement("path", {
    fill: "#00247d",
    d: "M0 0h1200v600H0z"
  }), React__default.createElement("path", {
    stroke: "#fff",
    d: "M0 0l600 300M0 300L600 0",
    strokeWidth: 60,
    clipPath: "url(#sa3487802)"
  }), React__default.createElement("path", {
    stroke: "#cc142b",
    d: "M0 0l600 300M0 300L600 0",
    strokeWidth: 40,
    clipPath: "url(#sa841c70d)"
  }), React__default.createElement("path", {
    stroke: "#fff",
    d: "M300 0v300M0 150h600",
    strokeWidth: 100,
    clipPath: "url(#sa3487802)"
  }), React__default.createElement("path", {
    stroke: "#cc142b",
    d: "M300 0v300M0 150h600",
    strokeWidth: 60,
    clipPath: "url(#sa3487802)"
  }), React__default.createElement("g", null, React__default.createElement("use", {
    xlinkHref: "#s4ed22915",
    fill: "#fff",
    transform: "matrix(45.4 0 0 45.4 900 120)"
  }), React__default.createElement("use", {
    xlinkHref: "#s4ed22915",
    fill: "#cc142b",
    transform: "matrix(30 0 0 30 900 120)"
  }), React__default.createElement("g", {
    transform: "rotate(82 900 240)"
  }, React__default.createElement("use", {
    xlinkHref: "#s4ed22915",
    fill: "#fff",
    transform: "rotate(-82 519.022 -457.666) scale(40.4)"
  }), React__default.createElement("use", {
    xlinkHref: "#s4ed22915",
    fill: "#cc142b",
    transform: "rotate(-82 519.022 -457.666) scale(25)"
  })), React__default.createElement("g", {
    transform: "rotate(82 900 240)"
  }, React__default.createElement("use", {
    xlinkHref: "#s4ed22915",
    fill: "#fff",
    transform: "rotate(-82 668.57 -327.666) scale(45.4)"
  }), React__default.createElement("use", {
    xlinkHref: "#s4ed22915",
    fill: "#cc142b",
    transform: "rotate(-82 668.57 -327.666) scale(30)"
  })), React__default.createElement("g", null, React__default.createElement("use", {
    xlinkHref: "#s4ed22915",
    fill: "#fff",
    transform: "matrix(50.4 0 0 50.4 900 480)"
  }), React__default.createElement("use", {
    xlinkHref: "#s4ed22915",
    fill: "#cc142b",
    transform: "matrix(35 0 0 35 900 480)"
  }))));
};

module.exports = FlagNz;
