// react-country-flags/dist/cjs/FlagUz.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagUz = function FlagUz(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 500 250"
  }, props), React__default.createElement("path", {
    fill: "#1eb53a",
    d: "M0 0h500v250H0z"
  }), React__default.createElement("path", {
    fill: "#0099b5",
    d: "M0 0h500v125H0z"
  }), React__default.createElement("path", {
    fill: "#ce1126",
    d: "M0 80h500v90H0z"
  }), React__default.createElement("path", {
    fill: "#fff",
    d: "M0 85h500v80H0z"
  }), React__default.createElement("circle", {
    cx: 70,
    cy: 40,
    r: 30,
    fill: "#fff"
  }), React__default.createElement("circle", {
    cx: 80,
    cy: 40,
    r: 30,
    fill: "#0099b5"
  }), React__default.createElement("g", {
    fill: "#fff",
    transform: "translate(136 64)"
  }, React__default.createElement("g", {
    id: "s81505487"
  }, React__default.createElement("g", {
    id: "s0cdc4632"
  }, React__default.createElement("g", {
    id: "se3b5ffcb"
  }, React__default.createElement("g", {
    id: "saa261654"
  }, React__default.createElement("path", {
    id: "sab6fe2ef",
    d: "M0-6v6h3",
    transform: "rotate(18 0 -6)"
  }), React__default.createElement("use", {
    xlinkHref: "#sab6fe2ef",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#saa261654",
    transform: "rotate(72)"
  })), React__default.createElement("use", {
    xlinkHref: "#saa261654",
    transform: "rotate(-72)"
  }), React__default.createElement("use", {
    xlinkHref: "#se3b5ffcb",
    transform: "rotate(144)"
  })), React__default.createElement("use", {
    xlinkHref: "#s0cdc4632",
    y: -24
  }), React__default.createElement("use", {
    xlinkHref: "#s0cdc4632",
    y: -48
  })), React__default.createElement("use", {
    xlinkHref: "#s81505487",
    x: 24
  }), React__default.createElement("use", {
    xlinkHref: "#s81505487",
    x: 48
  }), React__default.createElement("use", {
    xlinkHref: "#s0cdc4632",
    x: -48
  }), React__default.createElement("use", {
    xlinkHref: "#s0cdc4632",
    x: -24
  }), React__default.createElement("use", {
    xlinkHref: "#s0cdc4632",
    x: -24,
    y: -24
  })));
};

module.exports = FlagUz;
