// react-country-flags/dist/cjs/FlagBa.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagBa = function FlagBa(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 16 8"
  }, props), React__default.createElement("path", {
    fill: "#002395",
    d: "M0 0h16v8H0z"
  }), React__default.createElement("path", {
    d: "M4.24 0h8v8z",
    fill: "#fecb00"
  }), React__default.createElement("g", {
    id: "sbcf6d303"
  }, React__default.createElement("path", {
    d: "M2.353.525L2.8-.85 3.247.525l-1.17-.85h1.446z",
    fill: "#fff",
    id: "s637b9754"
  }), React__default.createElement("use", {
    xlinkHref: "#s637b9754",
    x: 1,
    y: 1
  }), React__default.createElement("use", {
    xlinkHref: "#s637b9754",
    x: 2,
    y: 2
  })), React__default.createElement("use", {
    xlinkHref: "#sbcf6d303",
    x: 3,
    y: 3
  }), React__default.createElement("use", {
    xlinkHref: "#sbcf6d303",
    x: 6,
    y: 6
  }));
};

module.exports = FlagBa;
