// react-country-flags/dist/cjs/FlagMn.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagMn = function FlagMn(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    viewBox: "0 0 4800 2400"
  }, props), React__default.createElement("g", {
    fill: "#f9cf02"
  }, React__default.createElement("g", {
    transform: "scale(5)"
  }, React__default.createElement("path", {
    fill: "#c4272f",
    d: "M0 0h960v480H0z"
  }), React__default.createElement("path", {
    fill: "#015197",
    d: "M320 0h320v480H320z"
  }), React__default.createElement("circle", {
    cx: 160,
    cy: 164,
    r: 44
  }), React__default.createElement("circle", {
    cx: 160,
    cy: 144,
    r: 48,
    fill: "#c4272f"
  }), React__default.createElement("circle", {
    cx: 160,
    cy: 152,
    r: 32
  }), React__default.createElement("path", {
    d: "M140 92a20 20 0 0 0 40 0c0-8-5-9-5-12s3-7-3-12c3 5-2 6-2 11s2 5 2 9a4 4 0 0 1-8 0c0-4 4-8 4-14s-1-8-4-13-8-9-4-13c-6 2-3 10-3 15s-4 8-4 14 3 7 3 11a4 4 0 0 1-8 0c0-4 2-4 2-9s-5-6-2-11c-6 5-3 9-3 12s-5 4-5 12zM72 216v192h40V216zm136 0v192h40V216zm-88 32v16h80v-16zm0 112v16h80v-16zm0-144h80l-40 24zm0 168h80l-40 24z"
  })), React__default.createElement("g", {
    stroke: "#c4272f",
    strokeWidth: 24
  }, React__default.createElement("circle", {
    cx: 800,
    cy: 1560,
    r: 212
  }), React__default.createElement("path", {
    fill: "none",
    d: "M800 1348a106 106 0 0 1 0 212 106 106 0 0 0 0 212"
  }))), React__default.createElement("g", {
    fill: "#c4272f"
  }, React__default.createElement("circle", {
    cx: 800,
    cy: 1454,
    r: 40
  }), React__default.createElement("circle", {
    cx: 800,
    cy: 1666,
    r: 40
  })));
};

module.exports = FlagMn;
