// react-country-flags/dist/cjs/FlagVn.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagVn = function FlagVn(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "-15 -10 30 20"
  }, props), React__default.createElement("path", {
    fill: "#DA251d",
    d: "M-20-15h40v30h-40z"
  }), React__default.createElement("g", {
    id: "sbd7c3fad",
    transform: "translate(0 -6)"
  }, React__default.createElement("path", {
    id: "sf915d7b8",
    fill: "#FF0",
    transform: "rotate(18)",
    d: "M0 0v6h4"
  }), React__default.createElement("use", {
    xlinkHref: "#sf915d7b8",
    transform: "scale(-1 1)"
  })), React__default.createElement("g", {
    id: "sca97a1a1",
    transform: "rotate(72)"
  }, React__default.createElement("use", {
    xlinkHref: "#sbd7c3fad"
  }), React__default.createElement("use", {
    xlinkHref: "#sbd7c3fad",
    transform: "rotate(72)"
  })), React__default.createElement("use", {
    xlinkHref: "#sca97a1a1",
    transform: "scale(-1 1)"
  }));
};

module.exports = FlagVn;
