// react-country-flags/dist/cjs/FlagSb.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagSb = function FlagSb(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 800 400"
  }, props), React__default.createElement("svg", {
    viewBox: "0 0 800 400"
  }, React__default.createElement("path", {
    d: "M0 400V0h800z",
    fill: "#0051ba"
  }), React__default.createElement("path", {
    d: "M0 400h800V0z",
    fill: "#215b33"
  }), React__default.createElement("path", {
    d: "M0 400L800 0",
    stroke: "#fcd116",
    strokeWidth: 36
  }), React__default.createElement("g", {
    transform: "translate(140 120)"
  }, React__default.createElement("g", {
    id: "sa931cb16",
    fill: "#fff"
  }, React__default.createElement("g", {
    id: "s0ff378c8"
  }, React__default.createElement("g", {
    id: "s7f7678ea"
  }, React__default.createElement("path", {
    id: "sd3ee06bb",
    d: "M0-40V0h20z",
    transform: "rotate(18 0 -40)"
  }), React__default.createElement("use", {
    xlinkHref: "#sd3ee06bb",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s7f7678ea",
    transform: "rotate(72)"
  })), React__default.createElement("use", {
    xlinkHref: "#s7f7678ea",
    transform: "rotate(-72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s0ff378c8",
    transform: "rotate(144)"
  })), React__default.createElement("g", {
    id: "s9554276b",
    transform: "rotate(40.6)"
  }, React__default.createElement("use", {
    id: "s89fe461e",
    xlinkHref: "#sa931cb16",
    x: -104,
    transform: "rotate(-40.6 -104 0)"
  }), React__default.createElement("use", {
    xlinkHref: "#s89fe461e",
    x: 208
  })), React__default.createElement("use", {
    xlinkHref: "#s9554276b",
    transform: "scale(-1 1)"
  }))));
};

module.exports = FlagSb;
