// react-country-flags/dist/cjs/FlagUm.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagUm = function FlagUm(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 7410 3900"
  }, props), React__default.createElement("path", {
    fill: "#b22234",
    d: "M0 0h7410v3900H0z"
  }), React__default.createElement("path", {
    d: "M0 450h7410m0 600H0m0 600h7410m0 600H0m0 600h7410m0 600H0",
    stroke: "#fff",
    strokeWidth: 300
  }), React__default.createElement("path", {
    fill: "#3c3b6e",
    d: "M0 0h2964v2100H0z"
  }), React__default.createElement("g", {
    fill: "#fff"
  }, React__default.createElement("g", {
    id: "sd492708b"
  }, React__default.createElement("g", {
    id: "sb046aaaf"
  }, React__default.createElement("g", {
    id: "s320a5d8f"
  }, React__default.createElement("g", {
    id: "s67032ec6"
  }, React__default.createElement("path", {
    id: "s23dcff17",
    d: "M247 90l70.534 217.082-184.66-134.164h228.253L176.466 307.082z"
  }), React__default.createElement("use", {
    xlinkHref: "#s23dcff17",
    y: 420
  }), React__default.createElement("use", {
    xlinkHref: "#s23dcff17",
    y: 840
  }), React__default.createElement("use", {
    xlinkHref: "#s23dcff17",
    y: 1260
  })), React__default.createElement("use", {
    xlinkHref: "#s23dcff17",
    y: 1680
  })), React__default.createElement("use", {
    xlinkHref: "#s67032ec6",
    x: 247,
    y: 210
  })), React__default.createElement("use", {
    xlinkHref: "#sb046aaaf",
    x: 494
  })), React__default.createElement("use", {
    xlinkHref: "#sd492708b",
    x: 988
  }), React__default.createElement("use", {
    xlinkHref: "#sb046aaaf",
    x: 1976
  }), React__default.createElement("use", {
    xlinkHref: "#s320a5d8f",
    x: 2470
  })));
};

module.exports = FlagUm;
