// react-country-flags/dist/cjs/FlagSn.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagSn = function FlagSn(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 900 600"
  }, props), React__default.createElement("path", {
    fill: "#00853f",
    d: "M0 0h300v600H0z"
  }), React__default.createElement("path", {
    fill: "#fdef42",
    d: "M300 0h300v600H300z"
  }), React__default.createElement("path", {
    fill: "#e31b23",
    d: "M600 0h300v600H600z"
  }), React__default.createElement("g", {
    transform: "translate(450 300)",
    fill: "#00853f"
  }, React__default.createElement("g", {
    id: "s0ba0da37"
  }, React__default.createElement("path", {
    id: "sdc6318e3",
    d: "M0-100V0h50z",
    transform: "rotate(18 0 -100)"
  }), React__default.createElement("use", {
    xlinkHref: "#sdc6318e3",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s0ba0da37",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s0ba0da37",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#s0ba0da37",
    transform: "rotate(216)"
  }), React__default.createElement("use", {
    xlinkHref: "#s0ba0da37",
    transform: "rotate(288)"
  })));
};

module.exports = FlagSn;
