// react-country-flags/dist/cjs/FlagCl.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagCl = function FlagCl(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "-4 -4 24 16"
  }, props), React__default.createElement("path", {
    d: "M20 12H-4V-4h24z",
    fill: "#d52b1e"
  }), React__default.createElement("path", {
    d: "M4 4h16v-8H-4z",
    fill: "#fff"
  }), React__default.createElement("path", {
    d: "M4 4h-8v-8h8z",
    fill: "#0039a6"
  }), React__default.createElement("g", {
    id: "s39d4d252"
  }, React__default.createElement("g", {
    id: "sd7c065a0"
  }, React__default.createElement("path", {
    d: "M0-2v2h1z",
    fill: "#fff",
    transform: "rotate(18 0 -2)",
    id: "s2b94775c"
  }), React__default.createElement("use", {
    xlinkHref: "#s2b94775c",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#sd7c065a0",
    transform: "rotate(72)"
  })), React__default.createElement("use", {
    xlinkHref: "#sd7c065a0",
    transform: "rotate(-72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s39d4d252",
    transform: "rotate(144)"
  }));
};

module.exports = FlagCl;
