// react-country-flags/dist/cjs/FlagSo.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagSo = function FlagSo(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 81 54"
  }, props), React__default.createElement("path", {
    fill: "#4189DD",
    d: "M0 0h81v54H0z"
  }), React__default.createElement("g", {
    transform: "matrix(13 0 0 13 40.5 27)"
  }, React__default.createElement("g", {
    id: "seea529c4"
  }, React__default.createElement("path", {
    id: "s529b48d0",
    fill: "#FFF",
    transform: "rotate(18 3.157 -.5)",
    d: "M0 0v1h.5z"
  }), React__default.createElement("use", {
    xlinkHref: "#s529b48d0",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#seea529c4",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#seea529c4",
    transform: "rotate(-72)"
  }), React__default.createElement("use", {
    xlinkHref: "#seea529c4",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#seea529c4",
    transform: "rotate(-144)"
  })));
};

module.exports = FlagSo;
