// react-country-flags/dist/cjs/FlagAn.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagAn = function FlagAn(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "-27 -14 54 36"
  }, props), React__default.createElement("path", {
    fill: "#fff",
    d: "M-27-14h54v36h-54z"
  }), React__default.createElement("path", {
    fill: "#dc171d",
    d: "M-6-14H6v36H-6z"
  }), React__default.createElement("path", {
    fill: "#012a87",
    d: "M-27-2h54v12h-54z"
  }), React__default.createElement("g", {
    id: "s67c630d5",
    fill: "#fff",
    transform: "scale(1.5)"
  }, React__default.createElement("g", {
    id: "s9818fb11"
  }, React__default.createElement("path", {
    id: "s87e52a8c",
    transform: "rotate(18 3.157 -.5)",
    d: "M0 0v1h.5z"
  }), React__default.createElement("use", {
    xlinkHref: "#s87e52a8c",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s9818fb11",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s9818fb11",
    transform: "rotate(-72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s9818fb11",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#s9818fb11",
    transform: "rotate(-144)"
  })), React__default.createElement("g", {
    id: "s20030a88"
  }, React__default.createElement("use", {
    xlinkHref: "#s67c630d5",
    transform: "translate(8 3.314)"
  }), React__default.createElement("use", {
    xlinkHref: "#s67c630d5",
    transform: "translate(3.314 8)"
  })), React__default.createElement("use", {
    xlinkHref: "#s20030a88",
    transform: "scale(-1 1)"
  }));
};

module.exports = FlagAn;
