// react-country-flags/dist/cjs/FlagPa.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagPa = function FlagPa(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 36 24"
  }, props), React__default.createElement("path", {
    fill: "#fff",
    d: "M0 0h36v24H0z"
  }), React__default.createElement("g", {
    fill: "#005293"
  }, React__default.createElement("g", {
    id: "s95443cde",
    transform: "translate(9 6)"
  }, React__default.createElement("g", {
    id: "s22632413"
  }, React__default.createElement("path", {
    id: "s94eef583",
    d: "M0-3v3h1.5z",
    transform: "rotate(18 0 -3)"
  }), React__default.createElement("use", {
    xlinkHref: "#s94eef583",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s22632413",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s22632413",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#s22632413",
    transform: "rotate(216)"
  }), React__default.createElement("use", {
    xlinkHref: "#s22632413",
    transform: "rotate(288)"
  })), React__default.createElement("path", {
    d: "M0 12h18v12H0z"
  })), React__default.createElement("g", {
    fill: "#d21034"
  }, React__default.createElement("path", {
    d: "M18 0h18v12H18z"
  }), React__default.createElement("use", {
    xlinkHref: "#s95443cde",
    x: 18,
    y: 12
  })));
};

module.exports = FlagPa;
