// react-country-flags/dist/cjs/FlagKp.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagKp = function FlagKp(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 72 36"
  }, props), React__default.createElement("path", {
    fill: "#024FA2",
    d: "M0 0h72v36H0z"
  }), React__default.createElement("path", {
    fill: "#fff",
    d: "M0 6h72v24H0z"
  }), React__default.createElement("path", {
    fill: "#ED1C27",
    d: "M0 7h72v22H0z"
  }), React__default.createElement("circle", {
    fill: "#fff",
    cx: 24,
    cy: 18,
    r: 8
  }), React__default.createElement("g", {
    transform: "matrix(7.75 0 0 7.75 24 18)",
    fill: "#ED1C27"
  }, React__default.createElement("g", {
    id: "s0f34304b"
  }, React__default.createElement("path", {
    id: "s535ea555",
    transform: "rotate(18 3.157 -.5)",
    d: "M0 0v1h.5z"
  }), React__default.createElement("use", {
    xlinkHref: "#s535ea555",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s0f34304b",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s0f34304b",
    transform: "rotate(-72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s0f34304b",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#s0f34304b",
    transform: "rotate(-144)"
  })));
};

module.exports = FlagKp;
