// react-country-flags/dist/cjs/FlagSy.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagSy = function FlagSy(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 180 120"
  }, props), React__default.createElement("path", {
    d: "M0 0h180v120H0z"
  }), React__default.createElement("path", {
    fill: "#fff",
    d: "M0 0h180v80H0z"
  }), React__default.createElement("path", {
    fill: "#ce1126",
    d: "M0 0h180v40H0z"
  }), React__default.createElement("g", {
    id: "s3f41d733",
    fill: "#007a3d"
  }, React__default.createElement("g", {
    id: "see16fe3e"
  }, React__default.createElement("g", {
    id: "s2bec85f8"
  }, React__default.createElement("path", {
    d: "M54 47v13h8",
    transform: "rotate(18 54 47)",
    id: "s9ec60cac"
  }), React__default.createElement("use", {
    xlinkHref: "#s9ec60cac",
    x: -108,
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s2bec85f8",
    transform: "rotate(72 54 60)"
  })), React__default.createElement("use", {
    xlinkHref: "#s2bec85f8",
    transform: "rotate(-72 54 60)"
  }), React__default.createElement("use", {
    xlinkHref: "#see16fe3e",
    transform: "rotate(144 54 60)"
  })), React__default.createElement("use", {
    xlinkHref: "#s3f41d733",
    x: 72
  }));
};

module.exports = FlagSy;
