// react-country-flags/dist/cjs/FlagKr.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagKr = function FlagKr(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "-36 -24 72 48"
  }, props), React__default.createElement("path", {
    fill: "#fff",
    d: "M-36-24h72v48h-72z"
  }), React__default.createElement("g", {
    transform: "rotate(-56.31)"
  }, React__default.createElement("g", {
    id: "s3f8924d6"
  }, React__default.createElement("path", {
    id: "sf14b4394",
    d: "M-6-25H6m-12 3H6m-12 3H6",
    stroke: "#000",
    strokeWidth: 2
  }), React__default.createElement("use", {
    xlinkHref: "#sf14b4394",
    y: 44
  })), React__default.createElement("path", {
    stroke: "#fff",
    d: "M0 17v10"
  }), React__default.createElement("circle", {
    fill: "#c60c30",
    r: 12
  }), React__default.createElement("path", {
    fill: "#003478",
    d: "M0-12A6 6 0 0 0 0 0a6 6 0 0 1 0 12 12 12 0 0 1 0-24z"
  })), React__default.createElement("g", {
    transform: "rotate(-123.69)"
  }, React__default.createElement("use", {
    xlinkHref: "#s3f8924d6"
  }), React__default.createElement("path", {
    stroke: "#fff",
    d: "M0-23.5v3M0 17v3.5m0 3v3"
  })));
};

module.exports = FlagKr;
