// react-country-flags/dist/cjs/FlagGw.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagGw = function FlagGw(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "-2 -3 12 6"
  }, props), React__default.createElement("path", {
    fill: "#ce1126",
    d: "M-2-3h4v6h-4z"
  }), React__default.createElement("path", {
    fill: "#fcd116",
    d: "M2-3h8v3H2z"
  }), React__default.createElement("path", {
    fill: "#009e49",
    d: "M2 0h8v3H2z"
  }), React__default.createElement("g", {
    id: "sb56d3e06"
  }, React__default.createElement("path", {
    id: "sfbfd31ce",
    d: "M0-1v1h.5",
    transform: "rotate(18 0 -1)",
    fill: "#000"
  }), React__default.createElement("use", {
    xlinkHref: "#sfbfd31ce",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#sb56d3e06",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#sb56d3e06",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#sb56d3e06",
    transform: "rotate(216)"
  }), React__default.createElement("use", {
    xlinkHref: "#sb56d3e06",
    transform: "rotate(288)"
  }));
};

module.exports = FlagGw;
