// react-country-flags/dist/cjs/FlagCw.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagCw = function FlagCw(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 54 36"
  }, props), React__default.createElement("path", {
    fill: "#002b7f",
    d: "M0 0h54v36H0z"
  }), React__default.createElement("path", {
    d: "M0 22.5h54V27H0z",
    fill: "#f9e814"
  }), React__default.createElement("g", {
    fill: "#fff",
    id: "s493bbfcf"
  }, React__default.createElement("g", {
    id: "s6b1ec82d"
  }, React__default.createElement("g", {
    id: "s4581c9ca"
  }, React__default.createElement("path", {
    d: "M12 8v4h2z",
    transform: "rotate(18 12 8)",
    id: "sbbe6da30"
  }), React__default.createElement("use", {
    xlinkHref: "#sbbe6da30",
    x: -24,
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s4581c9ca",
    transform: "rotate(72 12 12)"
  })), React__default.createElement("use", {
    xlinkHref: "#s4581c9ca",
    transform: "rotate(-72 12 12)"
  }), React__default.createElement("use", {
    xlinkHref: "#s6b1ec82d",
    transform: "rotate(144 12 12)"
  })), React__default.createElement("use", {
    xlinkHref: "#s493bbfcf",
    x: -4,
    y: -4,
    transform: "scale(.75)"
  }));
};

module.exports = FlagCw;
