// react-country-flags/dist/cjs/FlagNa.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagNa = function FlagNa(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 90 60"
  }, props), React__default.createElement("svg", {
    viewBox: "0 0 90 60"
  }, React__default.createElement("path", {
    d: "M90 0H0v60z",
    fill: "#003580"
  }), React__default.createElement("path", {
    d: "M0 60h90V0z",
    fill: "#009543"
  }), React__default.createElement("path", {
    d: "M0 60L90 0",
    stroke: "#fff",
    strokeWidth: 20
  }), React__default.createElement("path", {
    d: "M0 60L90 0",
    stroke: "#d21034",
    strokeWidth: 15
  }), React__default.createElement("g", {
    fill: "#ffce00",
    transform: "translate(18 16.341)"
  }, React__default.createElement("g", {
    id: "s5c6eab9b"
  }, React__default.createElement("path", {
    id: "sb5936373",
    d: "M0 10l1.553-4.204h-3.106z"
  }), React__default.createElement("use", {
    xlinkHref: "#sb5936373",
    transform: "rotate(90)"
  }), React__default.createElement("use", {
    xlinkHref: "#sb5936373",
    transform: "rotate(180)"
  }), React__default.createElement("use", {
    xlinkHref: "#sb5936373",
    transform: "rotate(270)"
  })), React__default.createElement("use", {
    xlinkHref: "#s5c6eab9b",
    transform: "rotate(30)"
  }), React__default.createElement("use", {
    xlinkHref: "#s5c6eab9b",
    transform: "rotate(60)"
  }), React__default.createElement("circle", {
    r: 5.5,
    stroke: "#003580"
  }))));
};

module.exports = FlagNa;
