// react-country-flags/dist/cjs/FlagKm.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagKm = function FlagKm(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 500 300"
  }, props), React__default.createElement("path", {
    fill: "#3A75C4",
    d: "M0 0h500v300H0z"
  }), React__default.createElement("path", {
    fill: "#CE1126",
    d: "M0 0h500v225H0z"
  }), React__default.createElement("path", {
    fill: "#FFF",
    d: "M0 0h500v150H0z"
  }), React__default.createElement("path", {
    fill: "#FFC61E",
    d: "M0 0h500v75H0z"
  }), React__default.createElement("path", {
    fill: "#3D8E33",
    d: "M0 300l250-150L0 0v300z"
  }), React__default.createElement("circle", {
    fill: "#FFF",
    cx: 85,
    cy: 150,
    r: 67.5
  }), React__default.createElement("circle", {
    fill: "#3D8E33",
    cx: 115,
    cy: 150,
    r: 67.5
  }), React__default.createElement("path", {
    id: "s639b734d",
    fill: "#FFF",
    d: "M100.01 89.2l7.36 22.588-19.258-13.949h23.776L92.63 111.788l7.38-22.588z"
  }), React__default.createElement("use", {
    xlinkHref: "#s639b734d",
    y: 32.208
  }), React__default.createElement("use", {
    xlinkHref: "#s639b734d",
    y: 64.417
  }), React__default.createElement("use", {
    xlinkHref: "#s639b734d",
    y: 96.625
  }));
};

module.exports = FlagKm;
