// react-country-flags/dist/cjs/FlagWs.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagWs = function FlagWs(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 2880 1440"
  }, props), React__default.createElement("defs", null, React__default.createElement("path", {
    id: "s3bed8bf4",
    d: "M0 0l.309.951L-.5.363h1l-.809.588z"
  })), React__default.createElement("path", {
    fill: "#ce1126",
    d: "M0 0h2880v1440H0z"
  }), React__default.createElement("path", {
    fill: "#002b7f",
    d: "M0 0h1440v720H0z"
  }), React__default.createElement("g", {
    fill: "#fff"
  }, React__default.createElement("use", {
    xlinkHref: "#s3bed8bf4",
    transform: "matrix(160 0 0 160 720 30)"
  }), React__default.createElement("use", {
    xlinkHref: "#s3bed8bf4",
    transform: "matrix(160 0 0 160 500 210)"
  }), React__default.createElement("use", {
    xlinkHref: "#s3bed8bf4",
    transform: "matrix(150 0 0 150 945 180)"
  }), React__default.createElement("use", {
    xlinkHref: "#s3bed8bf4",
    transform: "matrix(100 0 0 100 830 364.894)"
  }), React__default.createElement("use", {
    xlinkHref: "#s3bed8bf4",
    transform: "matrix(200 0 0 200 720 499.789)"
  })));
};

module.exports = FlagWs;
