// react-country-flags/dist/cjs/FlagTw.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagTw = function FlagTw(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "-60 -40 240 160"
  }, props), React__default.createElement("rect", {
    x: -60,
    y: -40,
    width: "100%",
    height: "100%",
    fill: "#fe0000"
  }), React__default.createElement("rect", {
    x: -60,
    y: -40,
    width: "50%",
    height: "50%",
    fill: "#000095"
  }), React__default.createElement("path", {
    id: "s4c495e82",
    d: "M8 0L0 30-8 0l8-30M0 8l30-8L0-8l-30 8",
    fill: "#fff"
  }), React__default.createElement("use", {
    xlinkHref: "#s4c495e82",
    transform: "rotate(30)"
  }), React__default.createElement("use", {
    xlinkHref: "#s4c495e82",
    transform: "rotate(60)"
  }), React__default.createElement("circle", {
    r: 17,
    fill: "#000095"
  }), React__default.createElement("circle", {
    r: 15,
    fill: "#fff"
  }));
};

module.exports = FlagTw;
