// react-country-flags/dist/cjs/FlagZa.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagZa = function FlagZa(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 9 6"
  }, props), React__default.createElement("clipPath", {
    id: "s62718bce"
  }, React__default.createElement("path", {
    d: "M0 0l4.5 3L0 6",
    id: "s55eec673"
  })), React__default.createElement("clipPath", {
    id: "se7780f5a"
  }, React__default.createElement("path", {
    d: "M0 0h9v6H0z"
  })), React__default.createElement("g", {
    clipPath: "url(#se7780f5a)"
  }, React__default.createElement("path", {
    d: "M0 0v6h9V0z",
    fill: "#002395"
  }), React__default.createElement("path", {
    d: "M0 0v3h9V0z",
    fill: "#de3831"
  }), React__default.createElement("g", {
    strokeWidth: 2,
    stroke: "#fff"
  }, React__default.createElement("path", {
    d: "M0 0l4.5 3L0 6m4.5-3H9",
    id: "s549537da"
  }), React__default.createElement("use", {
    xlinkHref: "#s55eec673",
    stroke: "#ffb612",
    clipPath: "url(#s62718bce)"
  })), React__default.createElement("use", {
    xlinkHref: "#s549537da",
    fill: "none",
    stroke: "#007a4d",
    strokeWidth: 1.2
  })));
};

module.exports = FlagZa;
