// react-country-flags/dist/cjs/FlagMm.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagMm = function FlagMm(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 18 12"
  }, props), React__default.createElement("path", {
    fill: "#FECB00",
    d: "M0 0h18v6H0z"
  }), React__default.createElement("path", {
    fill: "#EA2839",
    d: "M0 6h18v6H0z"
  }), React__default.createElement("path", {
    fill: "#34B233",
    d: "M0 4h18v4H0z"
  }), React__default.createElement("g", {
    transform: "translate(9 6.422) scale(4.422)"
  }, React__default.createElement("path", {
    id: "s363654a1",
    fill: "#FFF",
    d: "M-.325 0L0-1l.325 1z"
  }), React__default.createElement("use", {
    xlinkHref: "#s363654a1",
    transform: "rotate(-144)"
  }), React__default.createElement("use", {
    xlinkHref: "#s363654a1",
    transform: "rotate(-72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s363654a1",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s363654a1",
    transform: "rotate(144)"
  })));
};

module.exports = FlagMm;
