// react-country-flags/dist/cjs/FlagCm.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagCm = function FlagCm(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 90 60"
  }, props), React__default.createElement("path", {
    fill: "#007a5e",
    d: "M0 0h30v60H0z"
  }), React__default.createElement("path", {
    fill: "#ce1126",
    d: "M30 0h30v60H30z"
  }), React__default.createElement("path", {
    fill: "#fcd116",
    d: "M60 0h30v60H60z"
  }), React__default.createElement("g", {
    transform: "translate(45 30)",
    fill: "#fcd116"
  }, React__default.createElement("g", {
    id: "sccbc82c5"
  }, React__default.createElement("path", {
    id: "sa455f754",
    d: "M0-8v8h4z",
    transform: "rotate(18 0 -8)"
  }), React__default.createElement("use", {
    xlinkHref: "#sa455f754",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#sccbc82c5",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#sccbc82c5",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#sccbc82c5",
    transform: "rotate(216)"
  }), React__default.createElement("use", {
    xlinkHref: "#sccbc82c5",
    transform: "rotate(288)"
  })));
};

module.exports = FlagCm;
