// react-country-flags/dist/cjs/FlagPh.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagPh = function FlagPh(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 180 90"
  }, props), React__default.createElement("path", {
    fill: "#0038a8",
    d: "M0 0h180v90H0z"
  }), React__default.createElement("path", {
    fill: "#ce1126",
    d: "M0 45h180v45H0z"
  }), React__default.createElement("path", {
    d: "M77.942 45L0 90V0",
    fill: "#fff"
  }), React__default.createElement("g", {
    transform: "translate(28 45)",
    fill: "#fcd116"
  }, React__default.createElement("circle", {
    r: 9
  }), React__default.createElement("g", {
    id: "s94ec592d"
  }, React__default.createElement("g", {
    id: "sf31cda82"
  }, React__default.createElement("g", {
    id: "sa25ec7ea"
  }, React__default.createElement("path", {
    d: "M-1 0l.062.062L0 0l-.938-.062z",
    transform: "scale(19)"
  }), React__default.createElement("path", {
    id: "s7c77c517",
    d: "M-.884.116l.05.05L0 0z",
    transform: "scale(19.2381)"
  }), React__default.createElement("use", {
    xlinkHref: "#s7c77c517",
    transform: "scale(1 -1)"
  })), React__default.createElement("use", {
    xlinkHref: "#sa25ec7ea",
    transform: "rotate(45)"
  })), React__default.createElement("use", {
    xlinkHref: "#sf31cda82",
    transform: "rotate(90)"
  })), React__default.createElement("use", {
    xlinkHref: "#s94ec592d",
    transform: "rotate(180)"
  }), React__default.createElement("g", {
    transform: "translate(-2.02)"
  }, React__default.createElement("g", {
    id: "s9adb718d",
    transform: "translate(37.962)"
  }, React__default.createElement("path", {
    id: "sfefc3cc8",
    d: "M5 0L1.618 1.176l-.073 3.58-2.163-2.854-3.427 1.037L-2 0z"
  }), React__default.createElement("use", {
    xlinkHref: "#sfefc3cc8",
    transform: "scale(1 -1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s9adb718d",
    transform: "rotate(120)"
  }), React__default.createElement("use", {
    xlinkHref: "#s9adb718d",
    transform: "rotate(-120)"
  }))));
};

module.exports = FlagPh;
