// react-country-flags/dist/cjs/FlagKn.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagKn = function FlagKn(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 750 500"
  }, props), React__default.createElement("svg", {
    viewBox: "0 0 750 500"
  }, React__default.createElement("path", {
    d: "M750 0H0v500",
    fill: "#009e49"
  }), React__default.createElement("path", {
    d: "M0 500h750V0",
    fill: "#ce1126"
  }), React__default.createElement("path", {
    d: "M0 500L750 0",
    stroke: "#fcd116",
    strokeWidth: 210
  }), React__default.createElement("path", {
    d: "M0 500L750 0",
    stroke: "#000",
    strokeWidth: 150
  }), React__default.createElement("g", {
    id: "sfc7af856",
    transform: "rotate(-33.69 514.716 -777.095)",
    fill: "#fff"
  }, React__default.createElement("g", {
    id: "s949522d6"
  }, React__default.createElement("path", {
    id: "s6930e572",
    d: "M0-70V0h35",
    transform: "rotate(18 0 -70)"
  }), React__default.createElement("use", {
    xlinkHref: "#s6930e572",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s949522d6",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s949522d6",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#s949522d6",
    transform: "rotate(216)"
  }), React__default.createElement("use", {
    xlinkHref: "#s949522d6",
    transform: "rotate(288)"
  })), React__default.createElement("use", {
    xlinkHref: "#sfc7af856",
    transform: "translate(-285 190)"
  })));
};

module.exports = FlagKn;
