// react-country-flags/dist/cjs/FlagNp.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagNp = function FlagNp(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "-17.582 -4.664 71.571 87.246"
  }, props), React__default.createElement("use", {
    xlinkHref: "#sab34580f",
    stroke: "#003893",
    strokeWidth: 5.165
  }), React__default.createElement("path", {
    id: "sab34580f",
    d: "M-15 37.574h60L-15 0v80h60l-60-60z",
    fill: "#DC143C"
  }), React__default.createElement("g", {
    fill: "#fff"
  }, React__default.createElement("path", {
    d: "M-11.95 23.483a12.84 12.84 0 0 0 23.9 0 11.95 11.95 0 0 1-23.9 0"
  }), React__default.createElement("g", {
    transform: "translate(0 29.045) scale(5.56106)"
  }, React__default.createElement("circle", {
    r: 1
  }), React__default.createElement("g", {
    id: "sdf2c85fb"
  }, React__default.createElement("g", {
    id: "se6643760"
  }, React__default.createElement("path", {
    id: "see3d2fc5",
    d: "M.195-.98L0-1.39l-.195.408",
    transform: "rotate(11.25)"
  }), React__default.createElement("use", {
    xlinkHref: "#see3d2fc5",
    transform: "rotate(22.5)"
  }), React__default.createElement("use", {
    xlinkHref: "#see3d2fc5",
    transform: "rotate(45)"
  })), React__default.createElement("use", {
    xlinkHref: "#se6643760",
    transform: "rotate(67.5)"
  })), React__default.createElement("use", {
    xlinkHref: "#sdf2c85fb",
    transform: "scale(-1 1)"
  })), React__default.createElement("g", {
    transform: "matrix(8.1434 0 0 8.1434 0 58.787)"
  }, React__default.createElement("circle", {
    r: 1
  }), React__default.createElement("g", {
    id: "s1830dce3"
  }, React__default.createElement("g", {
    id: "s3d3b3ac3"
  }, React__default.createElement("path", {
    id: "s36724cc8",
    d: "M.259.966L0 1.576l-.259-.61"
  }), React__default.createElement("use", {
    xlinkHref: "#s36724cc8",
    transform: "rotate(180)"
  })), React__default.createElement("use", {
    xlinkHref: "#s3d3b3ac3",
    transform: "rotate(90)"
  })), React__default.createElement("use", {
    xlinkHref: "#s1830dce3",
    transform: "rotate(30)"
  }), React__default.createElement("use", {
    xlinkHref: "#s1830dce3",
    transform: "rotate(60)"
  }))));
};

module.exports = FlagNp;
