// react-country-flags/dist/cjs/FlagGg.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagGg = function FlagGg(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "-18 -12 36 24"
  }, props), React__default.createElement("path", {
    fill: "#fff",
    d: "M-18-12h36v24h-36z"
  }), React__default.createElement("path", {
    d: "M0-12v24M-18 0h36",
    stroke: "#e8112d",
    strokeWidth: 6,
    fill: "none"
  }), React__default.createElement("path", {
    id: "sef8202b9",
    d: "M-9 2l1-1h9v-2h-9l-1-1z",
    fill: "#f9dd16"
  }), React__default.createElement("use", {
    xlinkHref: "#sef8202b9",
    transform: "rotate(90)"
  }), React__default.createElement("use", {
    xlinkHref: "#sef8202b9",
    transform: "rotate(-90)"
  }), React__default.createElement("use", {
    xlinkHref: "#sef8202b9",
    transform: "rotate(180)"
  }));
};

module.exports = FlagGg;
