// react-country-flags/dist/cjs/FlagHn.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagHn = function FlagHn(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 72 36"
  }, props), React__default.createElement("path", {
    fill: "#0073cf",
    d: "M0 0h72v36H0z"
  }), React__default.createElement("path", {
    fill: "#fff",
    d: "M0 12h72v12H0z"
  }), React__default.createElement("g", {
    id: "se30fab8f",
    transform: "matrix(2 0 0 2 36 18)",
    fill: "#0073cf"
  }, React__default.createElement("g", {
    id: "s1efae5c5"
  }, React__default.createElement("path", {
    id: "s0725a184",
    transform: "rotate(18 3.157 -.5)",
    d: "M0 0v1h.5z"
  }), React__default.createElement("use", {
    xlinkHref: "#s0725a184",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s1efae5c5",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s1efae5c5",
    transform: "rotate(-72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s1efae5c5",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#s1efae5c5",
    transform: "rotate(-144)"
  })), React__default.createElement("use", {
    xlinkHref: "#se30fab8f",
    transform: "translate(10 -3.2)"
  }), React__default.createElement("use", {
    xlinkHref: "#se30fab8f",
    transform: "translate(10 2.8)"
  }), React__default.createElement("use", {
    xlinkHref: "#se30fab8f",
    transform: "translate(-10 -3.2)"
  }), React__default.createElement("use", {
    xlinkHref: "#se30fab8f",
    transform: "translate(-10 2.8)"
  }));
};

module.exports = FlagHn;
