// react-country-flags/dist/cjs/FlagFm.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagFm = function FlagFm(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "-19 -10 38 20"
  }, props), React__default.createElement("path", {
    fill: "#75b2dd",
    d: "M-19-10h38v20h-38z"
  }), React__default.createElement("g", {
    id: "s17ec4d17",
    transform: "translate(0 -6)",
    fill: "#fff"
  }, React__default.createElement("g", {
    id: "sf00899f4"
  }, React__default.createElement("path", {
    id: "s0cf11e6f",
    d: "M0-2v2h1",
    transform: "rotate(18 0 -2)"
  }), React__default.createElement("use", {
    xlinkHref: "#s0cf11e6f",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#sf00899f4",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#sf00899f4",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#sf00899f4",
    transform: "rotate(216)"
  }), React__default.createElement("use", {
    xlinkHref: "#sf00899f4",
    transform: "rotate(288)"
  })), React__default.createElement("use", {
    xlinkHref: "#s17ec4d17",
    transform: "rotate(90)"
  }), React__default.createElement("use", {
    xlinkHref: "#s17ec4d17",
    transform: "rotate(180)"
  }), React__default.createElement("use", {
    xlinkHref: "#s17ec4d17",
    transform: "rotate(270)"
  }));
};

module.exports = FlagFm;
