// react-country-flags/dist/cjs/FlagTg.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagTg = function FlagTg(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 809 500"
  }, props), React__default.createElement("path", {
    fill: "#006a4e",
    d: "M0 0h809v500H0z"
  }), React__default.createElement("path", {
    fill: "#ffce00",
    d: "M0 100h809v100H0zM0 300h809v100H0z"
  }), React__default.createElement("path", {
    fill: "#d21034",
    d: "M0 0h300v300H0z"
  }), React__default.createElement("g", {
    transform: "translate(150 150)",
    fill: "#fff"
  }, React__default.createElement("g", {
    id: "s07b6308c"
  }, React__default.createElement("path", {
    id: "s2c3db29c",
    d: "M0-95V0h50",
    transform: "rotate(18 0 -95)"
  }), React__default.createElement("use", {
    xlinkHref: "#s2c3db29c",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s07b6308c",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s07b6308c",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#s07b6308c",
    transform: "rotate(216)"
  }), React__default.createElement("use", {
    xlinkHref: "#s07b6308c",
    transform: "rotate(288)"
  })));
};

module.exports = FlagTg;
