// react-country-flags/dist/cjs/FlagMw.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagMw = function FlagMw(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "-225 -114 450 300"
  }, props), React__default.createElement("path", {
    d: "M-225-114h450v300h-450z"
  }), React__default.createElement("g", {
    fill: "#CE1126"
  }, React__default.createElement("circle", {
    r: 64.5
  }), React__default.createElement("path", {
    d: "M-225-14h450V86h-450z"
  }), React__default.createElement("g", {
    id: "sd900dd45"
  }, React__default.createElement("g", {
    id: "s3c3a9268"
  }, React__default.createElement("g", {
    id: "s4f60582f"
  }, React__default.createElement("g", {
    id: "s5e4c7efc"
  }, React__default.createElement("path", {
    id: "s77ff1721",
    d: "M0-102c-2.2 0-3 3.3-3 6.5 0 8 1 12 3 23.5 2.04-11.5 3-15.6 3-23.5 0-3.3-1-6.5-3-6.5"
  }), React__default.createElement("use", {
    transform: "rotate(5)",
    xlinkHref: "#s77ff1721"
  })), React__default.createElement("use", {
    transform: "rotate(10)",
    xlinkHref: "#s5e4c7efc"
  })), React__default.createElement("use", {
    transform: "rotate(20)",
    xlinkHref: "#s4f60582f"
  })), React__default.createElement("use", {
    transform: "rotate(40)",
    xlinkHref: "#s3c3a9268"
  })), React__default.createElement("use", {
    transform: "rotate(-80)",
    xlinkHref: "#sd900dd45"
  })), React__default.createElement("path", {
    d: "M-225-21.5h450v7.5h-450z"
  }), React__default.createElement("path", {
    fill: "#339E35",
    d: "M-225 86h450v100h-450z"
  }));
};

module.exports = FlagMw;
