// react-country-flags/dist/cjs/FlagNu.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagNu = function FlagNu(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 120 60"
  }, props), React__default.createElement("svg", {
    viewBox: "2 3 120 60"
  }, React__default.createElement("defs", null, React__default.createElement("path", {
    id: "s052d0373",
    d: "M0 0v15h70v15H60L0 0zm0 30v10h30V0h30L0 30z"
  })), React__default.createElement("g", {
    fill: "none",
    fillRule: "evenodd"
  }, React__default.createElement("path", {
    fill: "#002868",
    fillRule: "nonzero",
    d: "M2 3h120v60H2z"
  }), React__default.createElement("path", {
    fill: "#000",
    fillRule: "nonzero",
    stroke: "#FFF",
    strokeWidth: 6,
    d: "M2 3l60 30m0-30L2 33"
  }), React__default.createElement("g", {
    transform: "translate(2 3)"
  }, React__default.createElement("mask", {
    id: "s4f4529e2",
    fill: "#fff"
  }, React__default.createElement("use", {
    xlinkHref: "#s052d0373"
  })), React__default.createElement("path", {
    fill: "#000",
    fillRule: "nonzero",
    stroke: "#CF142B",
    strokeWidth: 4,
    d: "M0 0l60 30m0-30L0 30",
    mask: "url(#b)"
  })), React__default.createElement("path", {
    fill: "#000",
    fillRule: "nonzero",
    stroke: "#FFF",
    strokeWidth: 10,
    d: "M32 3v40M2 18h70"
  }), React__default.createElement("path", {
    fill: "#000",
    fillRule: "nonzero",
    stroke: "#CF142B",
    strokeWidth: 6,
    d: "M32 3v40M2 18h70"
  }), React__default.createElement("path", {
    fill: "#FCD116",
    fillRule: "nonzero",
    d: "M62 3h60v60H2V33h60z"
  }), React__default.createElement("g", {
    fillRule: "nonzero",
    transform: "translate(11 5)"
  }, React__default.createElement("circle", {
    cx: 21,
    cy: 13,
    r: 5.104,
    fill: "#002868"
  }), React__default.createElement("path", {
    fill: "#FCD116",
    d: "M21 7.896l3 9.233-7.854-5.706h9.708L18 17.129M20.853.29l1.764 5.427L18 2.363h5.706L19.09 5.717m19.413 4.573l1.763 5.427-4.616-3.354h5.706l-4.616 3.354M21 20.29l1.763 5.427-4.616-3.354h5.706l-4.616 3.354M3.5 10.29l1.763 5.427-4.616-3.354h5.706l-4.616 3.354"
  })))));
};

module.exports = FlagNu;
