// react-country-flags/dist/cjs/FlagTl.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagTl = function FlagTl(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "-3 -6 24 12"
  }, props), React__default.createElement("path", {
    d: "M21 6V-6H-3V6z",
    fill: "#dc241f"
  }), React__default.createElement("path", {
    d: "M-3-6V6L9 0z",
    fill: "#ffc726"
  }), React__default.createElement("path", {
    d: "M-3-6V6l8-6z"
  }), React__default.createElement("g", {
    transform: "rotate(-26.565)"
  }, React__default.createElement("g", {
    id: "s21cd84c8"
  }, React__default.createElement("g", {
    id: "s74890357"
  }, React__default.createElement("path", {
    d: "M0-2.1V0h1z",
    fill: "#fff",
    transform: "rotate(18 0 -2.1)",
    id: "s152cec9a"
  }), React__default.createElement("use", {
    xlinkHref: "#s152cec9a",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s74890357",
    transform: "rotate(72)"
  })), React__default.createElement("use", {
    xlinkHref: "#s74890357",
    transform: "rotate(-72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s21cd84c8",
    transform: "rotate(144)"
  })));
};

module.exports = FlagTl;
