// react-country-flags/dist/cjs/FlagBi.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagBi = function FlagBi(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 500 300"
  }, props), React__default.createElement("defs", null, React__default.createElement("g", {
    id: "se6fdf592",
    fill: "#1eb53a"
  }, React__default.createElement("g", {
    id: "s0c0c4fed"
  }, React__default.createElement("g", {
    id: "sa8e2a64b"
  }, React__default.createElement("g", {
    id: "s25c8ce82"
  }, React__default.createElement("path", {
    id: "sc56da287",
    d: "M0-20V0h20",
    transform: "rotate(30 0 -20)"
  }), React__default.createElement("use", {
    xlinkHref: "#sc56da287",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s25c8ce82",
    transform: "rotate(120)"
  }), React__default.createElement("use", {
    xlinkHref: "#s25c8ce82",
    transform: "rotate(240)"
  })), React__default.createElement("use", {
    xlinkHref: "#sa8e2a64b",
    transform: "rotate(180)"
  })), React__default.createElement("use", {
    xlinkHref: "#s0c0c4fed",
    fill: "#ce1126",
    transform: "scale(.82)"
  })), React__default.createElement("use", {
    id: "se40402f7",
    xlinkHref: "#se6fdf592",
    x: 250,
    y: 106
  })), React__default.createElement("path", {
    d: "M0 0h500L0 300h500z",
    fill: "#ce1126"
  }), React__default.createElement("path", {
    d: "M0 0v300L500 0v300z",
    fill: "#1eb53a"
  }), React__default.createElement("path", {
    d: "M0 0l500 300m0-300L0 300",
    stroke: "#fff",
    strokeWidth: 40
  }), React__default.createElement("circle", {
    cx: 250,
    cy: 150,
    r: 85,
    fill: "#fff"
  }), React__default.createElement("use", {
    xlinkHref: "#se40402f7"
  }), React__default.createElement("use", {
    xlinkHref: "#se40402f7",
    transform: "rotate(120 250 150)"
  }), React__default.createElement("use", {
    xlinkHref: "#se40402f7",
    transform: "rotate(240 250 150)"
  }));
};

module.exports = FlagBi;
