// react-country-flags/dist/cjs/FlagSt.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagSt = function FlagSt(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 2800 1400"
  }, props), React__default.createElement("path", {
    fill: "#12ad2b",
    d: "M0 0h2800v1400H0z"
  }), React__default.createElement("path", {
    fill: "#ffce00",
    d: "M0 400h2800v600H0z"
  }), React__default.createElement("path", {
    d: "M0 0v1400l700-700",
    fill: "#d21034"
  }), React__default.createElement("g", {
    id: "s90568f6a",
    transform: "translate(1400 700)",
    fill: "#000"
  }, React__default.createElement("g", {
    id: "s8c3f5a9e"
  }, React__default.createElement("path", {
    id: "s2f1447cc",
    d: "M0-200V0h100",
    transform: "rotate(18 0 -200)"
  }), React__default.createElement("use", {
    xlinkHref: "#s2f1447cc",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s8c3f5a9e",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s8c3f5a9e",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#s8c3f5a9e",
    transform: "rotate(216)"
  }), React__default.createElement("use", {
    xlinkHref: "#s8c3f5a9e",
    transform: "rotate(288)"
  })), React__default.createElement("use", {
    xlinkHref: "#s90568f6a",
    x: 700
  }));
};

module.exports = FlagSt;
