// react-country-flags/dist/cjs/FlagBf.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagBf = function FlagBf(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 900 600"
  }, props), React__default.createElement("path", {
    fill: "#009e49",
    d: "M0 0h900v600H0z"
  }), React__default.createElement("path", {
    fill: "#ef2b2d",
    d: "M0 0h900v300H0z"
  }), React__default.createElement("g", {
    transform: "translate(450 300)",
    fill: "#fcd116"
  }, React__default.createElement("g", {
    id: "s71e1f406"
  }, React__default.createElement("path", {
    id: "s7d36ca28",
    d: "M0-100V0h50",
    transform: "rotate(18 0 -100)"
  }), React__default.createElement("use", {
    xlinkHref: "#s7d36ca28",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s71e1f406",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s71e1f406",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#s71e1f406",
    transform: "rotate(216)"
  }), React__default.createElement("use", {
    xlinkHref: "#s71e1f406",
    transform: "rotate(288)"
  })));
};

module.exports = FlagBf;
