// react-country-flags/dist/cjs/FlagLr.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagLr = function FlagLr(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 209 110"
  }, props), React__default.createElement("path", {
    fill: "#bf0a30",
    d: "M0 0h209v110H0z"
  }), React__default.createElement("path", {
    d: "M0 15h209M0 35h209M0 55h209M0 75h209M0 95h209",
    stroke: "#fff",
    strokeWidth: 10
  }), React__default.createElement("path", {
    fill: "#002868",
    d: "M0 0h50v50H0z"
  }), React__default.createElement("g", {
    transform: "matrix(15 0 0 15 25 25)"
  }, React__default.createElement("g", {
    id: "s5ef45845"
  }, React__default.createElement("path", {
    id: "s677b37c7",
    fill: "#fff",
    transform: "rotate(18 3.157 -.5)",
    d: "M0 0v1h.5z"
  }), React__default.createElement("use", {
    xlinkHref: "#s677b37c7",
    transform: "scale(-1 1)"
  })), React__default.createElement("use", {
    xlinkHref: "#s5ef45845",
    transform: "rotate(72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s5ef45845",
    transform: "rotate(-72)"
  }), React__default.createElement("use", {
    xlinkHref: "#s5ef45845",
    transform: "rotate(144)"
  }), React__default.createElement("use", {
    xlinkHref: "#s5ef45845",
    transform: "rotate(-144)"
  })));
};

module.exports = FlagLr;
