// react-country-flags/dist/cjs/FlagIn.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagIn = function FlagIn(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 225 150"
  }, props), React__default.createElement("path", {
    fill: "#f93",
    d: "M0 0h225v150H0z"
  }), React__default.createElement("path", {
    fill: "#fff",
    d: "M0 50h225v50H0z"
  }), React__default.createElement("path", {
    fill: "#128807",
    d: "M0 100h225v50H0z"
  }), React__default.createElement("g", {
    transform: "translate(112.5 75)"
  }, React__default.createElement("circle", {
    r: 20,
    fill: "#008"
  }), React__default.createElement("circle", {
    r: 17.5,
    fill: "#fff"
  }), React__default.createElement("circle", {
    r: 3.5,
    fill: "#008"
  }), React__default.createElement("g", {
    id: "s802f1c83"
  }, React__default.createElement("g", {
    id: "sd204b3bc"
  }, React__default.createElement("g", {
    id: "s19e0389f"
  }, React__default.createElement("g", {
    id: "s3810a7b3",
    fill: "#008"
  }, React__default.createElement("circle", {
    r: 0.875,
    transform: "rotate(7.5 -8.75 133.5)"
  }), React__default.createElement("path", {
    d: "M0 17.5L.6 7 0 2l-.6 5L0 17.5z"
  })), React__default.createElement("use", {
    xlinkHref: "#s3810a7b3",
    transform: "rotate(15)"
  })), React__default.createElement("use", {
    xlinkHref: "#s19e0389f",
    transform: "rotate(30)"
  })), React__default.createElement("use", {
    xlinkHref: "#sd204b3bc",
    transform: "rotate(60)"
  })), React__default.createElement("use", {
    xlinkHref: "#s802f1c83",
    transform: "rotate(120)"
  }), React__default.createElement("use", {
    xlinkHref: "#s802f1c83",
    transform: "rotate(-120)"
  })));
};

module.exports = FlagIn;
