// react-country-flags/dist/cjs/FlagUs.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagUs = function FlagUs(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 7410 3900"
  }, props), React__default.createElement("path", {
    fill: "#b22234",
    d: "M0 0h7410v3900H0z"
  }), React__default.createElement("path", {
    d: "M0 450h7410m0 600H0m0 600h7410m0 600H0m0 600h7410m0 600H0",
    stroke: "#fff",
    strokeWidth: 300
  }), React__default.createElement("path", {
    fill: "#3c3b6e",
    d: "M0 0h2964v2100H0z"
  }), React__default.createElement("g", {
    fill: "#fff"
  }, React__default.createElement("g", {
    id: "s363ba562"
  }, React__default.createElement("g", {
    id: "s0b2b18c6"
  }, React__default.createElement("g", {
    id: "sa8aa7014"
  }, React__default.createElement("g", {
    id: "s98f083b9"
  }, React__default.createElement("path", {
    id: "sd3967715",
    d: "M247 90l70.534 217.082-184.66-134.164h228.253L176.466 307.082z"
  }), React__default.createElement("use", {
    xlinkHref: "#sd3967715",
    y: 420
  }), React__default.createElement("use", {
    xlinkHref: "#sd3967715",
    y: 840
  }), React__default.createElement("use", {
    xlinkHref: "#sd3967715",
    y: 1260
  })), React__default.createElement("use", {
    xlinkHref: "#sd3967715",
    y: 1680
  })), React__default.createElement("use", {
    xlinkHref: "#s98f083b9",
    x: 247,
    y: 210
  })), React__default.createElement("use", {
    xlinkHref: "#s0b2b18c6",
    x: 494
  })), React__default.createElement("use", {
    xlinkHref: "#s363ba562",
    x: 988
  }), React__default.createElement("use", {
    xlinkHref: "#s0b2b18c6",
    x: 1976
  }), React__default.createElement("use", {
    xlinkHref: "#sa8aa7014",
    x: 2470
  })));
};

module.exports = FlagUs;
