// react-country-flags/dist/cjs/FlagCn.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagCn = function FlagCn(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 30 20"
  }, props), React__default.createElement("defs", null, React__default.createElement("path", {
    id: "sd8bd4f91",
    d: "M0-1L.588.809-.952-.309H.952L-.588.809z",
    fill: "#ffde00"
  })), React__default.createElement("path", {
    fill: "#de2910",
    d: "M0 0h30v20H0z"
  }), React__default.createElement("use", {
    xlinkHref: "#sd8bd4f91",
    transform: "matrix(3 0 0 3 5 5)"
  }), React__default.createElement("use", {
    xlinkHref: "#sd8bd4f91",
    transform: "rotate(23.036 .093 25.536)"
  }), React__default.createElement("use", {
    xlinkHref: "#sd8bd4f91",
    transform: "rotate(45.87 1.273 16.18)"
  }), React__default.createElement("use", {
    xlinkHref: "#sd8bd4f91",
    transform: "rotate(69.945 .996 12.078)"
  }), React__default.createElement("use", {
    xlinkHref: "#sd8bd4f91",
    transform: "rotate(20.66 -19.689 31.932)"
  }));
};

module.exports = FlagCn;
