// react-country-flags/dist/cjs/FlagAg.js
'use strict';

function _interopDefault (ex) { return (ex && (typeof ex === 'object') && 'default' in ex) ? ex['default'] : ex; }

var __chunk_1 = require('./chunk-cf97c2b1.js');
var React = require('react');
var React__default = _interopDefault(React);

var FlagAg = function FlagAg(props) {
  return React__default.createElement("svg", __chunk_1._extends({
    xmlnsXlink: "http://www.w3.org/1999/xlink",
    viewBox: "0 0 138 92"
  }, props), React__default.createElement("path", {
    fill: "#fff",
    d: "M0 0h138v92H0z"
  }), React__default.createElement("path", {
    d: "M0 0h138v36H0z"
  }), React__default.createElement("g", {
    transform: "translate(69 36)"
  }, React__default.createElement("g", {
    id: "sf44718e4"
  }, React__default.createElement("path", {
    id: "sc66e9395",
    d: "M-30 0L0-5.742V5.742z",
    fill: "#fcd116"
  }), React__default.createElement("use", {
    xlinkHref: "#sc66e9395",
    transform: "rotate(22.5)"
  }), React__default.createElement("use", {
    xlinkHref: "#sc66e9395",
    transform: "rotate(45)"
  })), React__default.createElement("use", {
    xlinkHref: "#sf44718e4",
    transform: "rotate(67.5)"
  }), React__default.createElement("use", {
    xlinkHref: "#sf44718e4",
    transform: "rotate(135)"
  })), React__default.createElement("path", {
    fill: "#0072c6",
    d: "M0 36h138v20H0z"
  }), React__default.createElement("path", {
    d: "M0 0v92h68M138 0v92H68",
    fill: "#ce1126"
  }));
};

module.exports = FlagAg;
